---
title: "Metabarcoding_Weekly"
authors: "Evelyn Pain, Fatimah Bouderdaben, and Jimmy Erkens"
format: html
editor: visual
---

# Brief Github protocol

-   Install Git

-   Import in project via version control

-   You will have a project and all the files on Github will be pulled locally

-   Generate a personal access token on Github

-   Make your changes

-   Commit your changes (and write what you did in the commit box)

-   Stage the files you want to push up (this is just checking a box)

    -   If you need to pull anything, pull it (you'll be prompted)

-   If there are any hiccups, follow the instructions in the R terminal

-   Push!

-   Make sure to pull before you get started in the future!

# Background and goals

ELP:

Our internship team has 4 main goals:

1.  To gain familiarity with genomics/genetics literature and theory through the assigned readings

2.  To get practice with the lab skills necessary for extraction, PCR and sequencing (with the Oxford Nanopore MinION)

3.  To develop a working protocol to DNA metabarcode woolly fecal samples to detect insects (and plants?) in their diet

4.  To practice bioinformatics skills by identifying and comparing the species eaten by YTWMs and common woolly monkeys

# Revised Protocol

TBA

```         
- <input type="checkbox" unchecked> Venus</input>
- <input type="checkbox" checked> Mars</input>
```

# Feb 12 - Feb 18

TBA

Fati: Practice Pipetting
