<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Metabarcoding_Weekly</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="Metabarcoding_Weekly_files/libs/clipboard/clipboard.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/quarto.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/popper.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/anchor.min.js"></script>
<link href="Metabarcoding_Weekly_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Metabarcoding_Weekly_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Metabarcoding_Weekly</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Evelyn Pain, Fatimah Bouderdaben, and Jimmy Erkens </p>
          </div>
  </div>
    
  
    
  </div>
  

</header>

<section id="brief-github-protocol-jimmy" class="level1">
<h1>Brief Github protocol (Jimmy)</h1>
<p>JRE</p>
<ul>
<li><p>Install Git</p></li>
<li><p>Import in project via version control</p></li>
<li><p>You will have a project and all the files on Github will be pulled locally</p></li>
<li><p>Generate a personal access token on Github</p></li>
<li><p>Make your changes</p></li>
<li><p>Commit your changes (and write what you did in the commit box)</p></li>
<li><p>Stage the files you want to push up (this is just checking a box)</p>
<ul>
<li>If you need to pull anything, pull it (you‚Äôll be prompted)</li>
</ul></li>
<li><p>If there are any hiccups, follow the instructions in the R terminal (git merge ‚Äìabort is your best friend!)</p></li>
<li><p>Push!</p></li>
<li><p>Make sure to pull before you get started in the future!</p></li>
</ul>
</section>
<section id="background-and-goals" class="level1">
<h1>Background and goals</h1>
<p>ELP</p>
<p>Our internship team has 4 main goals:</p>
<ol type="1">
<li><p>To gain familiarity with genomics/genetics literature and theory through the assigned readings</p></li>
<li><p>To get practice with the lab skills necessary for extraction, PCR and sequencing (with the Oxford Nanopore MinION)</p></li>
<li><p>To develop a working protocol to DNA metabarcode woolly fecal samples to detect insects (and plants?) in their diet</p></li>
<li><p>To practice bioinformatics skills by identifying and comparing the species eaten by YTWMs and common woolly monkeys</p></li>
</ol>
</section>
<section id="smagl-microbiome-extractions-dneasy-powerlyzer-powersoil-kit" class="level1">
<h1><strong>SMAGL Microbiome Extractions: DNeasy PowerLyzer PowerSoil Kit</strong></h1>
<section id="authors-developed-for-the-smagl-by-melissa-zarate-schmitt-lab-based-on-a-template-provided-by-dr.-liz-mallott-washington-university-in-st.-louis" class="level2">
<h2 class="anchored" data-anchor-id="authors-developed-for-the-smagl-by-melissa-zarate-schmitt-lab-based-on-a-template-provided-by-dr.-liz-mallott-washington-university-in-st.-louis">Authors: Developed for the SMAGL by Melissa Zarate (Schmitt Lab) based on a template provided by Dr.&nbsp;Liz Mallott (Washington University in St.&nbsp;Louis)</h2>
</section>
</section>
<section id="section" class="level1">
<h1>____________________________________________________</h1>
</section>
<section id="important-notes" class="level1">
<h1>Important Notes</h1>
<ul>
<li><p>When conducting this protocol, always wear a suitable lab coat, disposable acrylamide gloves and protective goggles/glasses. For more information, consult the appropriate safety data sheets (SDSs) available from the product supplier.</p></li>
<li><p>To ensure proper sample identification, label sides and tops of the tubes. Processes used in this protocol may rub the labels off of the tubes, so it‚Äôs important to duplicate!</p></li>
<li><p>Always include a control sample of sterile/DNAse free water for every extraction so that you can detect potential contamination!</p></li>
<li><p>Perform all centrifugation steps at room temperature (15‚Äì25¬∞C).</p></li>
<li><p>If <strong>Solution C1</strong> has precipitated, heat at 60¬∞C until the precipitate dissolves.</p></li>
<li><p>Shake to mix <strong>Solution C4</strong> before use.</p></li>
</ul>
<p>__________________________________________________________________________</p>
</section>
<section id="protocol-last-updated-18feb24-by-cas" class="level1">
<h1>Protocol (last updated 18FEB24 by CAS)</h1>
<section id="required-equipment-and-reagents" class="level3">
<h3 class="anchored" data-anchor-id="required-equipment-and-reagents">Required Equipment and Reagents</h3>
<ul>
<li><p>In this protocol, equipment available in the <strong>DNeasy PowerLyzer PowerSoil Kit</strong> will be marked in <strong>blue</strong>. Equipment available in the SMAGL will be in <strong>black</strong>.</p></li>
<li><p>Biosafety cabinet (BSC) - remember to prepare it before getting started! [<strong>BSC</strong>]. Unless otherwise specified, all steps of this protocol should be performed in the BSC.</p></li>
<li><p>Microcentrifuge set to 10,000 rcf (not rpm) [<strong>Centrifuge Workstation</strong>]</p></li>
<li><p>Pipettes and pipette tips (sterile filter tips), prepare ahead to make sure you don‚Äôt run out! [Use filter tips; available on shelving above <strong>PCR</strong> and <strong>Quantitation Workstations</strong>; you should ONLY use the pipettes in the <strong>BSC</strong> and/or <strong>Extraction Workstation</strong> for extractions. The <strong>Quantitation Station</strong> has its own set of pipettes.]</p></li>
<li><p>Heat block or water bath set at 65C (this takes time to heat up; get it ready before starting the protocol) [<strong>Extraction Workstation</strong>]</p></li>
<li><p>Vortexer [<strong>Extraction Workstation</strong> or <strong>BSC</strong>]</p></li>
<li><p>Bead-beater (we use an alternative to the PowerLyzer homogenizer) [<strong>BSC</strong>]</p></li>
</ul>
<p>Prior to starting, UV all tubes, pipettes, tips, etc. in the PCR or BSC for 10 minutes</p>
<ul>
<li><p>1. Add 0.25 g of fecal matter to the <strong>PowerBead Tubes</strong>.</p></li>
<li><p>2. Add 750 ùúál of <strong>PowerBead Solution</strong> to the <strong>PowerBead Tubes</strong>.</p></li>
<li><p>3. Add 60 ùúál of <strong>Solution C1</strong> to tubes and vortex briefly. If <strong>Solution C1</strong> has precipitated, heat to 60¬∫C until precipitate has dissolved (in heat block or water bath).</p>
<ul>
<li><strong>Note</strong>: Vortexing mixes the components in the <strong>PowerBead Tube</strong> and begins to disperse the sample.</li>
</ul></li>
<li><p>4. Incubate <strong>PowerBead Tubes</strong> at 65¬∫C for <strong>30 min</strong> in <strong>heat block</strong> or <strong>water bath.</strong></p>
<ul>
<li>This may be done by adding the <strong>Extraction Workstation</strong> dry block to the <strong>BSC</strong>; if you choose to do this, please clean it and add it to the BSC while preparing the BSC prior to beginning the protocol].</li>
</ul></li>
<li><p>5. Secure tubes in the <strong>Bead Beater [BSC]</strong> and run for 5 minutes at 2000 RPM, pause for 30 seconds, and run again for 5 minutes at 2000 RPM. You may run these in evenly divided batches to initiate the subsequent centrifuge step with balanced samples.</p></li>
<li><p>6. <strong>Centrifuge</strong> tubes at 10,000 g (rcf) for 30 s. Make sure the <strong>PowerBead Tubes</strong> rotate freely in the centrifuge without rubbing.</p>
<ul>
<li>If 30 s is not long enough to pellet the sample, you may run for 3 min at the same speed.</li>
</ul></li>
<li><p>7. Transfer supernatant (~400-500 ùúál) to clean <strong>2 ml Collection Tube</strong> (you may still have chunks of poo at this point in the supernatant).</p></li>
<li><p>8. Add 250 ùúál of <strong>Solution C2</strong> to tubes and vortex for 5 s. Incubate at 4¬∫C for 10 min.</p>
<ul>
<li><strong>Note</strong>: <strong>Solution C2</strong> has Inhibitor Removal Technology (IRT). It contains a reagent that can precipitate non-DNA organic and inorganic material, including humic substances, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications.</li>
</ul></li>
<li><p>9. <strong>Centrifuge</strong> the tubes at 10,000 g (rcf) for 1 min.</p></li>
<li><p>10. Transfer up to 600 ùúál of supernatant to a clean <strong>2 ml Collection Tube</strong> (make sure to only transfer the liquid ‚Äì none of the pellet here!).</p>
<ul>
<li><strong>Note</strong>: The pellet at this point contains non-DNA organic and inorganic material, including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet.</li>
</ul></li>
<li><p>11. Add 200 ùúál of <strong>Solution C3</strong> to the tubes and <strong>vortex</strong> briefly. Incubate at 4¬∫C for 10 min (use the old mini-fridge for this step, below the <strong>PCR Workstation</strong>).</p>
<ul>
<li><strong>Note</strong>: <strong>Solution C3</strong> has Inhibitor Removal Technology (IRT) and is a second reagent to precipitate additional non-DNA organic and inorganic material including humic acid, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications.</li>
</ul></li>
<li><p>12. <strong>Centrifuge</strong> the tubes at 10,000 g (rcf) for 1 min.</p></li>
<li><p>13. Transfer up to 750 ùúál of supernatant to a clean <strong>2 ml Collection Tube</strong>. Once again, be careful to avoid the pellet!</p>
<ul>
<li><strong>Note</strong>: The pellet at this point contains non-DNA organic and inorganic material including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet.</li>
</ul></li>
<li><p>14. Add 1.2 ml of <strong>Solution C4</strong> to the supernatant and <strong>vortex</strong> for 5 s. Shake <strong>Solution C4</strong> by hand to mix prior to adding. Also, this will fill the <strong>Collection Tube</strong> to the brim, so be prepared.</p>
<ul>
<li><p><strong>Note: This contains guanidine salts, which is highly reactive. Do not mix or clean with bleach!!!</strong></p></li>
<li><p><strong>Note</strong>: Solution C4 is a high-concentration salt solution. Since DNA binds tightly to silica at high salt concentrations, this will adjust the DNA solution salt concentrations to allow binding of DNA, but not non-DNA organic and inorganic material that may still be present at low levels, to the <strong>Spin Filter</strong>.</p></li>
</ul></li>
<li><p>15a. First wash: Pipette 675 ùúál of the supernatant to a <strong>Spin Filter</strong> and centrifuge at 10,000 g (rcf) for 1 min. Discard the liquid flow through. Repeat until all supernatant has passed through the column. (This will take 3x washes through the column).</p>
<ul>
<li><strong>Note</strong>: DNA is selectively bound to the silica membrane in the MB Spin Column device in the high salt solution. Contaminants pass through the filter membrane, leaving only DNA bound to the membrane.</li>
</ul></li>
<li><p>15b. Second Wash: Pipette 675 ùúál of the supernatant to the same <strong>Spin Filter</strong> and centrifuge at 10,000 g (rcf) for 1 min. Discard the liquid flow through.</p></li>
<li><p>15c. Third Wash: Pipette 675 ùúál of the supernatant to the same <strong>Spin Filter</strong> and centrifuge at 10,000 g (rcf) for 1 min. Discard the liquid flow through.</p></li>
<li><p>16. Add 500 ùúál of <strong>Solution C5</strong> to the filter and centrifuge at 10,000 g (rcf) for 30 s. Discard the liquid flow through.</p>
<ul>
<li><strong>Note</strong>: <strong>Solution C5</strong> is an ethanol-based wash solution used to further clean the DNA that is bound to the silica filter membrane in the <strong>Spin Filter</strong>. This wash solution removes residual salt, humic acid, and other contaminants while allowing the DNA to stay bound to the silica membrane.</li>
</ul></li>
<li><p>17. Centrifuge the <strong>Spin Filter</strong> column a second time at 10,000 g (rcf) for 1 min.</p>
<ul>
<li><strong>Note</strong>: This flow-through fraction is non-DNA organic and inorganic waste removed from the silica <strong>Spin Filter</strong> membrane by the ethanol wash solution. The second spin removes residual <strong>Solution C5</strong> (ethanol wash solution). It is critical to remove all traces of wash solution because the ethanol in <strong>Solution C5</strong> can interfere with many downstream DNA applications, such as PCR and gel electrophoresis.</li>
</ul></li>
<li><p>18. Place <strong>Spin Filter</strong> into a clean and labeled <strong>2 ml Collection Tube</strong> (this is the tube going into the freezer, so make sure it‚Äôs labeled appropriately). Avoid splashing any <strong>Solution C5</strong> onto the <strong>Spin Filter</strong>.</p></li>
<li><p>19. Incubate <strong>Solution C6</strong> for 5 minutes at 65¬∫C. Then, add 100 ùúál of <strong>Solution C6</strong> to the center of the filter membrane.</p>
<ul>
<li><strong>Note</strong>: Placing <strong>Solution C6</strong> in the center of the small white membrane will make sure the entire membrane is wet. This will result in a more efficient and complete release of the DNA from the <strong>Spin Filter</strong> membrane. As <strong>Solution C6</strong> passes through the silica membrane, DNA that was bound in the presence of high salt is selectively released by <strong>Solution C6</strong> (which is actually 10 mM Tris), which lacks salt.</li>
</ul></li>
<li><p>20. Incubate samples at room temperature for 25 minutes. Then <strong>centrifuge</strong> at 10,000 g (rcf) for 30 sec, and finally discard the spin filter.</p></li>
<li><p>Your DNA is ready for use!</p></li>
<li><p>22. Measure the dsDNA concentration using the <strong>Quantus</strong>, and sample A260/A280 using the <strong>NanoDrop</strong> [<strong>Quantitation Workstation</strong>]. Record these details in your lab notebook.</p></li>
<li><p>Place the properly labeled DNA elutions in the -20¬∫C freezer. Be sure to record where in the freezer you‚Äôve placed them, and that they are all together in a single, clearly labeled container/rack.</p></li>
</ul>
</section>
<section id="smagl-anthropological-genetics-internship-extractions" class="level2">
<h2 class="anchored" data-anchor-id="smagl-anthropological-genetics-internship-extractions">SMAGL Anthropological Genetics Internship Extractions</h2>
<p>NOTES: Please record by each step, above, who has done which step and when. Check with Prof.&nbsp;Schmitt prior to beginning regarding which samples should be extracted. ALWAYS make sure that there are sufficient reagents and equipment before beginning a protocol!</p>
<p>ELP</p>
</section>
</section>
<section id="extraction-protocol-qiagen-stool-kit" class="level1">
<h1>Extraction Protocol (<strong>QIAGEN Stool Kit)</strong></h1>
<p>This protocol is modified from the protocol developed by Mel Zarate in the SMAGL lab following modifications based on published protocols by Tony Di Fiore (University of Texas at Austin)</p>
<section id="general-notes-before-starting" class="level2">
<h2 class="anchored" data-anchor-id="general-notes-before-starting">General notes before starting</h2>
<ul>
<li><p>Ensure that you are wearing suitable PPE: a lab coat, protective eye glasses and gloves (you can consult the appropriate safety data sheets from product suppliers if you have any doubts)</p></li>
<li><p>Turn on the biosafety cabinet (BSC) and let it run for a minimum of 15 minutes before starting the protocol</p></li>
<li><p>Turn on the water bath and set it to 70 ¬∞C</p></li>
<li><p>Turn on the heat block and set it to 56 ¬∞C</p></li>
<li><p>Make up a new batch of 10% bleach solution (1 part bleach to 9 parts distilled water) for the day and dispose of any from previous days. Be sure to appropriately label the vessel with date and contents!</p></li>
<li><p>Aliquot sufficient 96-100% <strong>ethanol</strong> (available in the <strong>Flammables Cabinet</strong>) to a smaller vessel for use later in the protocol. Be sure to appropriately label the vessel with date and contents!</p></li>
<li><p>Clean all pipettes and work surfaces (including inside the biosafety cabinet) first with the bleach solution, then distilled water, and finally with 70% ethanol</p></li>
<li><p>Ensure that you have sufficient supplies (i.e., autoclaved, sterile tubes, pipette tips, etc.) before beginning</p></li>
<li><p>UV all tubes, pipettes and pipette tips in either the BSC or the PCR hood for 10 minutes before starting (be careful to limit your exposure to the UV light!)</p></li>
<li><p>Removed samples to be extracted from the freezer and let them thaw on ice</p></li>
</ul>
</section>
<section id="notes-on-the-protocol" class="level2">
<h2 class="anchored" data-anchor-id="notes-on-the-protocol">Notes on the protocol</h2>
<ul>
<li><p>Make sure to label all tubes on both the top and sides (it is easy for labels to get rubbed off during the protocol)</p></li>
<li><p>Always include a control sample of sterile/DNAse free water for every extraction so that you can detect potential contamination!</p></li>
<li><p>Perform all centrifugation steps at room temperature (15‚Äì25 ¬∞C)</p></li>
<li><p>The 2 ml tubes used in step 4 should be wide enough to accommodate an InhibitEX Tablet. If it‚Äôs not wide enough, you may need to break up the tablet to insert</p></li>
</ul>
</section>
<section id="required-equipment-and-supplies" class="level2">
<h2 class="anchored" data-anchor-id="required-equipment-and-supplies">Required equipment and supplies</h2>
<ul>
<li>In this protocol, equipment/reagents available in the QIAGEN stool Kit will be marked in <span style="color:blue">blue</span>. Equipment and supplies available in the SMAGL will be in <span style="color:green">green</span>.</li>
<li>You will use the following pieces of equipment: a <span style="color:green">biosafety cabinet</span>, <span style="color:green">fridge</span> (located below the <strong>PCR workstation</strong>), <span style="color:green">pipettes</span> (the extraction set are located inside the <strong>BSC</strong> or at the <strong>extraction workstation</strong>), <span style="color:green">microcentrifuge</span> (you will need to use the largest centrifuge at the <strong>Centrifuge Station</strong> as this is the only one that goes this fast) set to 20,000 rcf (not rpm), a <span style="color:green">benchtop microcentrifuge</span> (located at the <strong>quantification workstation</strong>), a <span style="color:green">heating block</span> set to 56 ¬∞C, a <span style="color:green">waterbath</span> set to 70 ¬∞C (located at the <strong>extraction workstation</strong>), a <span style="color:green">vortexer</span> (located at the <strong>extraction workstation</strong> or inside <strong>BSC</strong>), a <span style="color:green">tube rocker</span> (located at the <strong>extraction workstation</strong>), and a <span style="color:green">bead-beater</span> (located in the <strong>BSC</strong>)</li>
<li>You will need <span style="color:green">pipette tips</span> (located on shelves above the <strong>PCR</strong> or <strong>quantification workstations</strong>). ONLY use filter tips!</li>
<li>You will need <span style="color:green">2 mL</span> and <span style="color:green">1.5 mL tubes</span> which must be <strong>autoclaved</strong> prior to use! You‚Äôll have to count ahead in the protocol based on how many samples you‚Äôre running and how many tubes you‚Äôll need to figure out the amount so that you‚Äôll be prepared</li>
<li>You will also need <span style="color:green">96-100 % ethanol</span></li>
</ul>
</section>
<section id="protocol" class="level2">
<h2 class="anchored" data-anchor-id="protocol">Protocol</h2>
<ol type="1">
<li><p>Label the same number of <span style="color:green">2ml tubes</span> as samples, plus one for a control</p></li>
<li><p>Vortex the fecal sample for a few seconds</p></li>
<li><p>Pipet 200 ùúál of fecal slurry into the tube (you will need to cut the tips off each pipette tip with a razor blade to make the point wide enough to be able to suck up the sample)</p></li>
<li><p>Add 1.6 ml of <span style="color:blue">Buffer ASL</span> to each tube (if the solution has precipitated, dissolve by incubating at 70 ¬∞C in the <span style="color:green">waterbath</span>)</p></li>
<li><p>Vortex each tube for 1 minute or until the sample is thoroughly homogenized (It is very important to vortex the samples thoroughly. This helps ensure maximum DNA concentration in the final eluate)</p></li>
<li><p>Incubate samples at 56 ¬∫C in the <span style="color:green">heating block</span> for 30-60 min</p></li>
<li><p>Incubate samples at room temperature in the tube rocker for 30 minutes</p></li>
<li><p>An <strong>optional</strong> step is to further incubate the samples in the <span style="color:green">heating block</span> at 23 ¬∫C <strong>overnight. THIS IS A POTENTIAL STOPPING POINT</strong></p></li>
<li><p>Increase the temperature of the <span style="color:green">heating block</span> to 70 ¬∫C</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute to pellet stool particles (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p></li>
<li><p>Pipet 1.4 ml of the supernatant into the new <span style="color:green">2 ml tubes</span> and discard the pellet</p></li>
<li><p>Add 1 <span style="color:blue">InhibitEX Tablet</span> or 500 ùúál of <span style="color:blue">liquid InhibitEX</span> (newer kits have liquid, older have tablets) to each sample</p></li>
<li><p>Vortex each sample immediately for 1 minute or until the <span style="color:blue">InhibitEX</span> is completely suspended</p></li>
<li><p>Incubate suspension for 1 min at room temperature to allow inhibitors to adsorb to the <span style="color:blue">InhibitEX</span> matrix</p></li>
<li><p>Label a new set of <span style="color:green">1.5 ml tubes</span></p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes to pellet stool particles and inhibitors bound to the <span style="color:blue">InhibitEX</span> matrix (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: for difficult samples centrifugation for 3 minutes may result in a pellet that is not sufficiently compact. Therefore it may be difficult to remove enough supernatant to transfer 600 ùúál supernatant after the next centrifugation step. In these cases, we recommend to centrifuge for 6 minutes</li>
<li><strong>Note</strong>: When processing more than 12 samples, for this step and step 17 we recommend processing batches of no more than 12 samples each. This is because the pellets formed after centrifugation will break up quickly if the supernatant is not removed immediately</li>
</ul></li>
<li><p>Immediately after the <span style="color:green">microcentrifuge</span> stops, pipet all of the supernatant into the clean <span style="color:green">1.5 ml tubes</span> and discard the pellet (Transferring small quantities of pelleted material from the previous step will not affect the procedure)</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p></li>
<li><p>Pipet 25 ùúál of <span style="color:blue">proteinase K</span> into the newly labeled tubes</p></li>
<li><p>Pipet 600 ùúál supernatant from step 18 into the tubes containing the <span style="color:blue">proteinase K</span></p></li>
<li><p>Add 600 ùúál <span style="color:blue">Buffer AL</span></p>
<ul>
<li><strong>Note</strong>: do <strong>NOT</strong> add <span style="color:blue">proteinase K</span> directly to <span style="color:blue">Buffer AL</span>. It is essential that the sample and <span style="color:blue">Buffer AL</span> are thoroughly mixed to form a homogeneous solution</li>
</ul></li>
<li><p>Vortex tubes for 15 seconds</p></li>
<li><p>Incubate the tubes in the <span style="color:green">heating block</span> at 70¬∞C for 60 min</p>
<ul>
<li>You may want to briefly centrifuge the tubes using the small <span style="color:green">benchtop centrifuge</span> at the <strong>Quantification Station</strong> to remove drops from the inside of the tube lid (make sure that the centrifuge is <strong>balanced</strong>)</li>
</ul></li>
<li><p>Add 600 ùúál of <span style="color:green">96‚Äì100 % ethanol</span> to the lysate</p></li>
<li><p>Vortex briefly</p>
<ul>
<li>You may want to briefly centrifuge the tubes using the small <span style="color:green">benchtop centrifuge</span> at the <strong>Quantification Station</strong> to remove drops from the inside of the tube lid (make sure that the centrifuge is <strong>balanced</strong>)</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">QIAamp spin columns</span> and <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Carefully apply 600 ùúál of the lysate to the <span style="color:blue">QIAamp spin columns</span> without moistening the rim and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span>, apply a second aliquot of 600 ùúál lysate and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span>, apply a third aliquot of 600 ùúál lysate and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span> and add 500 ùúál of <span style="color:blue">Buffer AW1</span> and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span> and add 500 ùúál of <span style="color:blue">Buffer AW2</span> and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: residual <span style="color:blue">Buffer AW2</span> in the eluate may cause problems in downstream applications. Some centrifuge rotors may vibrate upon deceleration, resulting in the flow-through, which contains <span style="color:blue">Buffer AW2</span>, contacting the <span style="color:blue">QIAamp spin column</span>. Removing the <span style="color:blue">QIAamp spin column</span> and <span style="color:blue">collection tube</span> from the rotor may also cause flow-through to come into contact with the <span style="color:blue">QIAamp spin column</span></li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: this step is optional but recommended help eliminate the chance of <span style="color:blue">Buffer AW2</span> carryover</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate (the filtrate should be minimal)</p></li>
<li><p>Place <span style="color:blue">Buffer AE</span> in the waterbath at 70 ¬∫C to incubate for 10 minutes</p></li>
<li><p>Label a new set of <span style="color:green">1.5 ml tubes</span> (These are the tubes going into the freezer, so labeling is extra important!)</p></li>
<li><p>Transfer the <span style="color:blue">QIAamp spin columns</span> into the new set of <span style="color:green">1.5 ml tubes</span></p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin columns</span>, pipet 100 ùúál of <span style="color:blue">Buffer AE</span> directly onto the QIAamp membrane (do not touch the membrane with the pipette tip), and close the cap</p></li>
<li><p>Incubate the tubes at room temperature for 25 minutes</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute to elute the DNA (you will need to use the largest centrifuge at the <strong>Centrifuge Station</strong> as this is the only one that goes this fast)</p></li>
<li><p>Pull the filter portion of the tube up and cap the tube (you can either discard it or save them if you want to try and get more DNA out of them)</p></li>
<li><p>The liquid in the tube contains the DNA from the samples and should be stored in the <span style="color:green">-20 ¬∞C freezer</span> when not in use! (Be sure to record where in the freezer you‚Äôve placed them, and make sure that they are all together in a single, clearly labeled container/rack)</p></li>
<li><p>The next step is to quantify the DNA in the samples (this is done by measuring the dsDNA concentration using the <span style="color:green">Quantus</span> and the sample A260/A280 using the <span style="color:green">Nanodrop</span>, both are located at the <strong>Extraction Workstation</strong>). The results should be recorded in your lab notebook</p>
<ul>
<li><strong>Note</strong>: DNA yield is typically 15‚Äì60 ùúág but, depending on the individual stool sample and the way it was stored, may range from 5 to 100 ùúág. DNA concentration is typically 75‚Äì300 ng/ùúál</li>
</ul></li>
</ol>
</section>
</section>
<section id="feb-12---feb-18" class="level1">
<h1>Feb 12 - Feb 18</h1>
<p>JRE</p>
<p>This was the week we were supposed to start the extraction protocol, but got postponed.</p>
<p>Fati: Practice Pipetting</p>
</section>
<section id="feb-20---feb-25" class="level1">
<h1>Feb 20 - Feb 25</h1>
<p><input type="checkbox" id="cbox3" value="third_checkbox"> <label for="cbox3"> Can I only select an ID? </label><br></p>
</section>
<section id="feb-26---mar-3" class="level1">
<h1>Feb 26 - Mar 3</h1>
<p>February 29, 2024:</p>
<p>ELP and JRE:</p>
<ul>
<li>UV‚Äôd 1000ùúál pipettes, 200ùúál pipettes, 12 2ml lo-bind tubes, razor blades, 200 and 1000 ùúál tips.</li>
<li>Set heatblock to 56C</li>
<li>ELP: made 10% bleach</li>
<li>ELP: made ice bath; put samples in ice bath</li>
<li>Wiped BSC down with 70% ethenol, water (twice), then 10% bleach</li>
</ul>
<p>Testing the 1000ml pipettes (ELP) (notetaker = FAB and JRE)</p>
<ul>
<li>They both look to be pipetting the same amount (marked with Sharpie)</li>
</ul>
<ol type="1">
<li>Pipet 200 ùúál of fecal slurry into the microcentrifuge tube. You may need to cut the end of the pipet tip with a razor blade, if necessary, to make pipetting easier. Remember to vortex the sample prior to pipetting, to make sure you are getting a good mixture of buffer and feces.</li>
</ol>
<ul>
<li>Label tubes (completed by JRE at 5:13pm)</li>
<li>Adding 200 micro l water to control tube (completed by JRE at 5:16pm)</li>
<li>Vortex sample VOYT-18-017 and pipette 200 micro l into tube 03 (completed by ELP at 5:18pm)</li>
<li>Vortex sample VOYT-18-036 and pipette almost 200 micro l into tube 04 (completed by ELP at 5:19)</li>
<li>Vortex sample GIUNCHI-18-001 and pipette 200 micro l into tube 05 (completed by ELP at 5:20)</li>
<li>Vortex sample DIFIORE-18-025 and pipette 200 micro l into tube 06 (completed by ELP at 5:25)</li>
<li>Vortex sample DIFIORE-18-031 and pipette 200 micro l into tube 09 (completed by JRE at 5:28)</li>
<li>Vortex sample DIFIORE-18-032 and pipette 200 micro l into tube 11 (completed by JRE at 5:33)</li>
<li>Vortex sample VOYT-18-029 and pipette 200 micro l into tube 02 (completed by JRE at 5:35)</li>
<li>Vortex sample GOMEZ-18-190 and pipette 200 micro l into tube 01 (completed by FAB at 5:39)</li>
<li>Vortex sample VOYT-18-006 and pipette 200 micro l into tube 10 (completed by FAB at 5:44)</li>
<li>Vortex sample DIFIORE-18-026 and pipette 200 micro l into tube 07 (completed by FAB at 5:46)</li>
<li>Vortex sample DIFIORE-18-027 and pipette 200 micro l into tube 08 (completed by FAB at 5:48)</li>
</ul>
<ol start="2" type="1">
<li>Add 1.6 ml Buffer ASL to each stool sample. Vortex continuously for 1 min or until the stool sample is thoroughly homogenized. Note: It is very important to vortex the samples thoroughly. This helps ensure maximum DNA concentration in the final eluate.</li>
</ol>
<ul>
<li><p>Pipette 800 micro l of Buffer ASL to each tube twice, changing pipette tips each time (using 2 pipette tips per tube). (ELP completed control, tube 03, tube 04, and tube 05 at 5:53; JRE completed tube 06, 09, 11[slightly less buffer was added], and 02 at 6:03; FAB completed tube 01, 10, 07, and 08 at 6:08)</p></li>
<li><p>Vortex each of the tubes for 1 minute (2 at a time, 1 in each hand); double check that tubes are closed</p></li>
<li><p>Completed in shifts by ELP, JRE, and FAB at 6:17.</p></li>
</ul>
<ol start="3" type="1">
<li>Incubate samples at 56¬∫C in the Dry Bath for 30-60 min, then for an additional 30 min on Tube Rocker. Optional: further incubate in the Dry Bath at 23 degrees C overnight. [2-29-24; 60 minutes JRE]</li>
</ol>
<ul>
<li>Started 56 degree C Dry Bath at 6:18 pm (completed at )</li>
<li>Started Tube Rocker at (completed at )</li>
<li>Left in overnight 23 degrees C Dry Bath at</li>
</ul>
<p>-Cleaned out BSC with 70% ethenol, water, then 10% bleach (completed by ELP and JRE at 6:27 pm)</p>
<table class="table">
<colgroup>
<col style="width: 17%">
<col style="width: 14%">
<col style="width: 14%">
<col style="width: 21%">
<col style="width: 16%">
<col style="width: 14%">
</colgroup>
<thead>
<tr class="header">
<th>Sample ID</th>
<th>Taxon</th>
<th>Label</th>
<th>Final Concentration (ng/ùúál)</th>
<th><strong>A260/A280 value</strong></th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Control (sterile water)</td>
<td>NA</td>
<td>AGI24-DBCL</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>GOMEZ-18-190</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB01</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-029</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB02</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>VOYT-18-017</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB03</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-036</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB04</td>
<td></td>
<td></td>
<td>Maybe not the entire 200ùúál</td>
</tr>
<tr class="even">
<td>GIUNCHI-18-001</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB05</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>DIFIORE-18-025</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB06</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>DIFIORE-18-026</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB07</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>DIFIORE-18-027</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB08</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>DIFIORE-18-031</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB09</td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-006</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB10</td>
<td></td>
<td></td>
<td>Pipetted most of 200 then a tiny bit extra some chunks</td>
</tr>
<tr class="even">
<td>DIFIORE-18-032</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB11</td>
<td></td>
<td></td>
<td>Slightly less buffer was added</td>
</tr>
</tbody>
</table>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "Óßã";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>