---
title: "Metabarcoding_Weekly"
authors: "Evelyn Pain, Fatimah Bouderdaben, and Jimmy Erkens"
format: html
editor: visual
---

# Brief Github protocol

-   Install Git

-   Import in project via version control

-   You will have a project and all the files on Github will be pulled locally

-   Generate a personal access token on Github

-   Make your changes

-   Commit your changes (and write what you did in the commit box)

-   Stage the files you want to push up (this is just checking a box)

    -   If you need to pull anything, pull it (you'll be prompted)

-   If there are any hiccups, follow the instructions in the R terminal

-   Push!

-   Make sure to pull before you get started in the future!

# Background and goals

ELP:

Our internship team has 4 main goals:

1.  To gain familiarity with genomics/genetics literature and theory through the assigned readings

2.  To get practice with the lab skills necessary for extraction, PCR and sequencing (with the Oxford Nanopore MinION)

3.  To develop a working protocol to DNA metabarcode woolly fecal samples to detect insects (and plants?) in their diet

4.  To practice bioinformatics skills by identifying and comparing the species eaten by YTWMs and common woolly monkeys

# Protocol (**DNeasy PowerLyzer PowerSoil Kit)**

ELP: This protocol is modified from the protocol developed by Mel Zarate in the SMAGL lab following a template from Dr. Liz Mallott (Washington University in St. Louis)

## General notes before starting

1.  Ensure that you are wearing suitable PPE: a lab coat, protective eye glasses and gloves (you can consult the appropriate safety data sheets from product suppliers if you have any doubts)

2.  Turn on the biosafety cabinet (BSC) and let it run for a minimum of 15 minutes before starting the protocol

3.  Turn on the water bath and set it to 65C

4.  Make up a new batch of 10% bleach solution (1 part bleach to 9 parts distilled water) for the day and dispose of any from previous days

5.  Clean all pipettes and work surfaces (including inside the biosafety cabinet) first with the bleach solution, then distilled water, and finally with 70% ethanol

6.  Ensure that you have sufficient supplies (i.e., autoclaved, sterile tubes, pipette tips, etc.) before beginning

7.  Removed samples to be extracted from the freezer and let them thaw on ice

## Notes on the protocol

-   Make sure to label all tubes on both the top and sides (it is easy for labels to get rubbed off during the protocol)

-   Always include a control sample of sterile/DNAse free water for every extraction so that you can detect potential contamination!

-   Perform all centrifugation steps at room temperature (15--25°C).

-   If **Solution C1** has precipitated, heat at 60°C until the precipitate dissolves.

-   Shake to mix **Solution C4** before use.

## Required equipment and supplies

-   In this protocol, equipment/reagents available in the DNeasy PowerLyzer PowerSoil Kit will be marked in [blue]{style="color:blue"}. Equipment and supplies available in the SMAGL will be in [green]{style="color:green"}.
-   You will use the following pieces of equipment: a [biosafety cabinet]{style="color:green"}, [pipettes]{style="color:green"} (the extraction set are located inside the **BSC** or at the **extraction workstation**), [microcentrifuge]{style="color:green"} (located at **centrifuge workstation**) set to 10,000 rcf (not rpm), a [heating block]{style="color:green"} or [waterbath]{style="color:green"} set to 65C (located at the **extraction workstation**), a [vortexer]{style="color:green"} (located at the **extraction workstation** or inside **BSC**) and a [bead-beater]{style="color:green"} (located in the **BSC**)
-   You will also need [pipette tips]{style="color:green"} (located on shelves above the **PCR** or **quantification workstations**). ONLY use filter tips!

```         
- <input type="checkbox" unchecked> Venus</input>
- <input type="checkbox" checked> Mars</input>
```

# Feb 12 - Feb 18

TBA

Fati: Practice Pipetting
