<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Metabarcoding_Weekly</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="Metabarcoding_Weekly_files/libs/clipboard/clipboard.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/quarto.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/popper.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/anchor.min.js"></script>
<link href="Metabarcoding_Weekly_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Metabarcoding_Weekly_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#brief-github-protocol" id="toc-brief-github-protocol" class="nav-link active" data-scroll-target="#brief-github-protocol">Brief Github protocol</a></li>
  <li><a href="#background-and-goals" id="toc-background-and-goals" class="nav-link" data-scroll-target="#background-and-goals">Background and goals</a></li>
  <li><a href="#extraction-protocol-qiagen-stool-kit" id="toc-extraction-protocol-qiagen-stool-kit" class="nav-link" data-scroll-target="#extraction-protocol-qiagen-stool-kit">Extraction Protocol (<strong>QIAGEN Stool Kit)</strong></a>
  <ul class="collapse">
  <li><a href="#general-notes-before-starting" id="toc-general-notes-before-starting" class="nav-link" data-scroll-target="#general-notes-before-starting">General notes before starting</a></li>
  <li><a href="#notes-on-the-protocol" id="toc-notes-on-the-protocol" class="nav-link" data-scroll-target="#notes-on-the-protocol">Notes on the protocol</a></li>
  <li><a href="#required-equipment-and-supplies" id="toc-required-equipment-and-supplies" class="nav-link" data-scroll-target="#required-equipment-and-supplies">Required equipment and supplies</a></li>
  <li><a href="#protocol" id="toc-protocol" class="nav-link" data-scroll-target="#protocol">Protocol</a></li>
  </ul></li>
  <li><a href="#quantus-quantitation-protocol" id="toc-quantus-quantitation-protocol" class="nav-link" data-scroll-target="#quantus-quantitation-protocol">Quantus Quantitation Protocol</a>
  <ul class="collapse">
  <li><a href="#important-notes" id="toc-important-notes" class="nav-link" data-scroll-target="#important-notes">Important Notes</a></li>
  <li><a href="#required-equipment" id="toc-required-equipment" class="nav-link" data-scroll-target="#required-equipment">Required Equipment</a></li>
  </ul></li>
  <li><a href="#feb-12---feb-18" id="toc-feb-12---feb-18" class="nav-link" data-scroll-target="#feb-12---feb-18">Feb 12 - Feb 18</a></li>
  <li><a href="#feb-20---feb-25" id="toc-feb-20---feb-25" class="nav-link" data-scroll-target="#feb-20---feb-25">Feb 20 - Feb 25</a></li>
  <li><a href="#feb-26---mar-3" id="toc-feb-26---mar-3" class="nav-link" data-scroll-target="#feb-26---mar-3">Feb 26 - Mar 3</a></li>
  <li><a href="#mar-4---mar-10" id="toc-mar-4---mar-10" class="nav-link" data-scroll-target="#mar-4---mar-10">Mar 4 - Mar 10</a>
  <ul class="collapse">
  <li><a href="#march-7-2024" id="toc-march-7-2024" class="nav-link" data-scroll-target="#march-7-2024">March 7, 2024</a></li>
  <li><a href="#march-8-2024" id="toc-march-8-2024" class="nav-link" data-scroll-target="#march-8-2024">March 8, 2024</a></li>
  </ul></li>
  <li><a href="#mar-25---mar-31" id="toc-mar-25---mar-31" class="nav-link" data-scroll-target="#mar-25---mar-31">Mar 25 - Mar 31</a>
  <ul class="collapse">
  <li><a href="#march-28-2024-post-aaba-re-quantitating" id="toc-march-28-2024-post-aaba-re-quantitating" class="nav-link" data-scroll-target="#march-28-2024-post-aaba-re-quantitating">March 28, 2024 (Post AABA re-quantitating)</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Metabarcoding_Weekly</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Evelyn Pain, Fatimah Bouderdaben, and Jimmy Erkens </p>
          </div>
  </div>
    
  
    
  </div>
  

</header>

<section id="brief-github-protocol" class="level1">
<h1>Brief Github protocol</h1>
<p>JRE</p>
<ul>
<li><p>Install Git</p></li>
<li><p>Import in project via version control</p></li>
<li><p>You will have a project and all the files on Github will be pulled locally</p></li>
<li><p>Generate a personal access token on Github</p></li>
<li><p>Make your changes</p></li>
<li><p>Commit your changes (and write what you did in the commit box)</p></li>
<li><p>Stage the files you want to push up (this is just checking a box)</p>
<ul>
<li>If you need to pull anything, pull it (you’ll be prompted)</li>
</ul></li>
<li><p>If there are any hiccups, follow the instructions in the R terminal (git merge –abort is your best friend!)</p></li>
<li><p>Push!</p></li>
<li><p>Make sure to pull before you get started in the future!</p></li>
</ul>
</section>
<section id="background-and-goals" class="level1">
<h1>Background and goals</h1>
<p>ELP</p>
<p>Our internship team has 4 main goals:</p>
<ol type="1">
<li><p>To gain familiarity with genomics/genetics literature and theory through the assigned readings</p></li>
<li><p>To get practice with the lab skills necessary for extraction, PCR and sequencing (with the Oxford Nanopore MinION)</p></li>
<li><p>To develop a working protocol to DNA metabarcode woolly fecal samples to detect insects (and plants?) in their diet</p></li>
<li><p>To practice bioinformatics skills by identifying and comparing the species eaten by YTWMs and common woolly monkeys</p></li>
</ol>
<p>ELP</p>
</section>
<section id="extraction-protocol-qiagen-stool-kit" class="level1">
<h1>Extraction Protocol (<strong>QIAGEN Stool Kit)</strong></h1>
<p>This protocol is modified from the protocol developed by Mel Zarate in the SMAGL lab following modifications based on published protocols by Tony Di Fiore (University of Texas at Austin)</p>
<section id="general-notes-before-starting" class="level2">
<h2 class="anchored" data-anchor-id="general-notes-before-starting">General notes before starting</h2>
<ul>
<li><p>Ensure that you are wearing suitable PPE: a lab coat, protective eye glasses and gloves (you can consult the appropriate safety data sheets from product suppliers if you have any doubts)</p></li>
<li><p>Turn on the biosafety cabinet (BSC) and let it run for a minimum of 15 minutes before starting the protocol</p></li>
<li><p>Turn on the water bath and set it to 70 °C</p></li>
<li><p>Turn on the heat block and set it to 56 °C</p></li>
<li><p>Make up a new batch of 10% bleach solution (1 part bleach to 9 parts distilled water) for the day and dispose of any from previous days. Be sure to appropriately label the vessel with date and contents!</p></li>
<li><p>Aliquot sufficient 96-100% <strong>ethanol</strong> (available in the <strong>Flammables Cabinet</strong>) to a smaller vessel for use later in the protocol. Be sure to appropriately label the vessel with date and contents!</p></li>
<li><p>Clean all pipettes and work surfaces (including inside the biosafety cabinet) first with the bleach solution, then distilled water, and finally with 70% ethanol</p></li>
<li><p>Ensure that you have sufficient supplies (i.e., autoclaved, sterile tubes, pipette tips, etc.) before beginning</p></li>
<li><p>UV all tubes, pipettes and pipette tips in either the BSC or the PCR hood for 10 minutes before starting (be careful to limit your exposure to the UV light!)</p></li>
<li><p>Removed samples to be extracted from the freezer and let them thaw on ice</p></li>
</ul>
</section>
<section id="notes-on-the-protocol" class="level2">
<h2 class="anchored" data-anchor-id="notes-on-the-protocol">Notes on the protocol</h2>
<ul>
<li><p>Make sure to label all tubes on both the top and sides (it is easy for labels to get rubbed off during the protocol)</p></li>
<li><p>Always include a control sample of sterile/DNAse free water for every extraction so that you can detect potential contamination!</p></li>
<li><p>Perform all centrifugation steps at room temperature (15–25 °C)</p></li>
<li><p>The 2 ml tubes used in step 4 should be wide enough to accommodate an InhibitEX Tablet. If it’s not wide enough, you may need to break up the tablet to insert</p></li>
</ul>
</section>
<section id="required-equipment-and-supplies" class="level2">
<h2 class="anchored" data-anchor-id="required-equipment-and-supplies">Required equipment and supplies</h2>
<ul>
<li>In this protocol, equipment/reagents available in the QIAGEN stool Kit will be marked in <span style="color:blue">blue</span>. Equipment and supplies available in the SMAGL will be in <span style="color:green">green</span>.</li>
<li>You will use the following pieces of equipment: a <span style="color:green">biosafety cabinet</span>, <span style="color:green">fridge</span> (located below the <strong>PCR workstation</strong>), <span style="color:green">pipettes</span> (the extraction set are located inside the <strong>BSC</strong> or at the <strong>extraction workstation</strong>), <span style="color:green">microcentrifuge</span> (you will need to use the largest centrifuge at the <strong>Centrifuge Station</strong> as this is the only one that goes this fast) set to 20,000 rcf (not rpm), a <span style="color:green">benchtop microcentrifuge</span> (located at the <strong>quantification workstation</strong>), a <span style="color:green">heating block</span> set to 56 °C, a <span style="color:green">waterbath</span> set to 70 °C (located at the <strong>extraction workstation</strong>), a <span style="color:green">vortexer</span> (located at the <strong>extraction workstation</strong> or inside <strong>BSC</strong>), a <span style="color:green">tube rocker</span> (located at the <strong>extraction workstation</strong>), and a <span style="color:green">bead-beater</span> (located in the <strong>BSC</strong>)</li>
<li>You will need <span style="color:green">pipette tips</span> (located on shelves above the <strong>PCR</strong> or <strong>quantification workstations</strong>). ONLY use filter tips!</li>
<li>You will need <span style="color:green">2 mL</span> and <span style="color:green">1.5 mL tubes</span> which must be <strong>autoclaved</strong> prior to use! You’ll have to count ahead in the protocol based on how many samples you’re running and how many tubes you’ll need to figure out the amount so that you’ll be prepared</li>
<li>You will also need <span style="color:green">96-100 % ethanol</span></li>
</ul>
</section>
<section id="protocol" class="level2">
<h2 class="anchored" data-anchor-id="protocol">Protocol</h2>
<ol type="1">
<li><p>Label the same number of <span style="color:green">2ml tubes</span> as samples, plus one for a control</p></li>
<li><p>Vortex the fecal sample for a few seconds</p></li>
<li><p>Pipet 200 𝜇l of fecal slurry into the tube (you will need to cut the tips off each pipette tip with a razor blade to make the point wide enough to be able to suck up the sample)</p></li>
<li><p>Add 1.6 ml of <span style="color:blue">Buffer ASL</span> to each tube (if the solution has precipitated, dissolve by incubating at 70 °C in the <span style="color:green">waterbath</span>)</p></li>
<li><p>Vortex each tube for 1 minute or until the sample is thoroughly homogenized (It is very important to vortex the samples thoroughly. This helps ensure maximum DNA concentration in the final eluate)</p></li>
<li><p>Incubate samples at 56 ºC in the <span style="color:green">heating block</span> for 30-60 min</p></li>
<li><p>Incubate samples at room temperature in the tube rocker for 30 minutes</p></li>
<li><p>An <strong>optional</strong> step is to further incubate the samples in the <span style="color:green">heating block</span> at 23 ºC <strong>overnight. THIS IS A POTENTIAL STOPPING POINT</strong></p></li>
<li><p>Increase the temperature of the <span style="color:green">heating block</span> to 70 ºC</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute to pellet stool particles (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p></li>
<li><p>Pipet 1.4 ml of the supernatant into the new <span style="color:green">2 ml tubes</span> and discard the pellet</p></li>
<li><p>Add 1 <span style="color:blue">InhibitEX Tablet</span> or 500 𝜇l of <span style="color:blue">liquid InhibitEX</span> (newer kits have liquid, older have tablets) to each sample</p></li>
<li><p>Vortex each sample immediately for 1 minute or until the <span style="color:blue">InhibitEX</span> is completely suspended</p></li>
<li><p>Incubate suspension for 1 min at room temperature to allow inhibitors to adsorb to the <span style="color:blue">InhibitEX</span> matrix</p></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes to pellet stool particles and inhibitors bound to the <span style="color:blue">InhibitEX</span> matrix (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: for difficult samples centrifugation for 3 minutes may result in a pellet that is not sufficiently compact. Therefore it may be difficult to remove enough supernatant to transfer 600 𝜇l supernatant after the next centrifugation step. In these cases, we recommend to centrifuge for 6 minutes</li>
<li><strong>Note</strong>: When processing more than 12 samples, for this step and step 17 we recommend processing batches of no more than 12 samples each. This is because the pellets formed after centrifugation will break up quickly if the supernatant is not removed immediately</li>
</ul></li>
<li><p>Immediately after the <span style="color:green">microcentrifuge</span> stops, pipet all of the supernatant into the clean <span style="color:green">2 ml tubes</span> and discard the pellet (Transferring small quantities of pelleted material from the previous step will not affect the procedure)</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p></li>
<li><p>Pipet 25 𝜇l of <span style="color:blue">proteinase K</span> into the newly labeled tubes</p></li>
<li><p>Pipet 600 𝜇l supernatant from step 18 into the tubes containing the <span style="color:blue">proteinase K</span></p></li>
<li><p>Add 600 𝜇l <span style="color:blue">Buffer AL</span></p>
<ul>
<li><strong>Note</strong>: do <strong>NOT</strong> add <span style="color:blue">proteinase K</span> directly to <span style="color:blue">Buffer AL</span>. It is essential that the sample and <span style="color:blue">Buffer AL</span> are thoroughly mixed to form a homogeneous solution</li>
</ul></li>
<li><p>Vortex tubes for 15 seconds</p></li>
<li><p>Incubate the tubes in the <span style="color:green">heating block</span> at 70°C for 60 min</p>
<ul>
<li>You may want to briefly centrifuge the tubes using the small <span style="color:green">benchtop centrifuge</span> at the <strong>Quantification Station</strong> to remove drops from the inside of the tube lid (make sure that the centrifuge is <strong>balanced</strong>)</li>
</ul></li>
<li><p>Add 600 𝜇l of <span style="color:green">96–100 % ethanol</span> to the lysate</p></li>
<li><p>Vortex briefly</p>
<ul>
<li>You may want to briefly centrifuge the tubes using the small <span style="color:green">benchtop centrifuge</span> at the <strong>Quantification Station</strong> to remove drops from the inside of the tube lid (make sure that the centrifuge is <strong>balanced</strong>)</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">QIAamp spin columns</span> and <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Carefully apply 600 𝜇l of the lysate to the <span style="color:blue">QIAamp spin columns</span> without moistening the rim and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Carefully pull the <span style="color:blue">QIAamp spin columns</span> out of the <span style="color:blue">collection tubes</span> and discard the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin columns</span> are empty</li>
</ul></li>
<li><p>Replace the <span style="color:blue">QIAamp spin columns</span> back into the now empty <span style="color:blue">collection tubes</span></p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span>, apply a second aliquot of 600 𝜇l lysate and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Carefully pull the <span style="color:blue">QIAamp spin columns</span> out of the <span style="color:blue">collection tubes</span> and discard the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
</ul></li>
<li><p>Replace the <span style="color:blue">QIAamp spin columns</span> back into the now empty <span style="color:blue">collection tubes</span></p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span>, apply a third aliquot of 600 𝜇l lysate and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span> and add 500 𝜇l of <span style="color:blue">Buffer AW1</span> and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span> and add 500 𝜇l of <span style="color:blue">Buffer AW2</span> and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: residual <span style="color:blue">Buffer AW2</span> in the eluate may cause problems in downstream applications. Some centrifuge rotors may vibrate upon deceleration, resulting in the flow-through, which contains <span style="color:blue">Buffer AW2</span>, contacting the <span style="color:blue">QIAamp spin column</span>. Removing the <span style="color:blue">QIAamp spin column</span> and <span style="color:blue">collection tube</span> from the rotor may also cause flow-through to come into contact with the <span style="color:blue">QIAamp spin column</span></li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: this step is optional but recommended help eliminate the chance of <span style="color:blue">Buffer AW2</span> carryover</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate (the filtrate should be minimal)</p></li>
<li><p>Place <span style="color:blue">Buffer AE</span> in the waterbath at 70 ºC to incubate for 10 minutes</p></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span> (These are the tubes going into the freezer, so labeling is extra important!)</p></li>
<li><p>Transfer the <span style="color:blue">QIAamp spin columns</span> into the new set of <span style="color:green">2 ml tubes</span></p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin columns</span>, pipet 100 𝜇l of <span style="color:blue">Buffer AE</span> directly onto the QIAamp membrane (do not touch the membrane with the pipette tip), and close the cap</p></li>
<li><p>Incubate the tubes at room temperature for 25 minutes</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute to elute the DNA (you will need to use the largest centrifuge at the <strong>Centrifuge Station</strong> as this is the only one that goes this fast)</p></li>
<li><p>Pull the filter portion of the tube up and cap the tube (you can either discard it or save them if you want to try and get more DNA out of them)</p></li>
<li><p>The liquid in the tube contains the DNA from the samples and should be stored in the <span style="color:green">-20 °C freezer</span> when not in use! (Be sure to record where in the freezer you’ve placed them, and make sure that they are all together in a single, clearly labeled container/rack)</p></li>
<li><p>The next step is to quantify the DNA in the samples (this is done by measuring the dsDNA concentration using the <span style="color:green">Quantus</span> and the sample A260/A280 using the <span style="color:green">Nanodrop</span>, both are located at the <strong>Extraction Workstation</strong>). The results should be recorded in your lab notebook</p>
<ul>
<li><strong>Note</strong>: DNA yield is typically 15–60 𝜇g but, depending on the individual stool sample and the way it was stored, may range from 5 to 100 𝜇g. DNA concentration is typically 75–300 ng/𝜇l</li>
</ul></li>
</ol>
</section>
</section>
<section id="quantus-quantitation-protocol" class="level1">
<h1>Quantus Quantitation Protocol</h1>
<p>Manufacturer’s protocol, without modifications from Melissa Zarate.</p>
<section id="important-notes" class="level2">
<h2 class="anchored" data-anchor-id="important-notes">Important Notes</h2>
<ul>
<li><p>Warm all assay components to room temperature before use.</p></li>
<li><p>You may need to prepare 1X TE buffer from more concentrated stock, although 1X TE Buffer is also available in the cabinets above the Centrifuge Workstation, and may already be pre-prepared in the Underbench Fridge.</p></li>
<li><p>If you have very little sample available, you may run this protocol using 1 μl rather than 2 μl.</p></li>
<li><p>Before running a tube in the Quantus, remember to wipe the tube down with Kimwipes to make sure the tube is clear.</p></li>
</ul>
</section>
<section id="required-equipment" class="level2">
<h2 class="anchored" data-anchor-id="required-equipment">Required Equipment</h2>
<ul>
<li><p>QuantiFluor® dsDNA System (Cat.# E2670) [<strong>Underbench Fridge - PCR Workstation</strong>], elements available in this kit will be noted in <strong>bold blue</strong>, elements that are available in the lab will be noted in <strong>bold black</strong>.</p></li>
<li><p>QuantusTM Fluorometer (Cat.# E6150) [<strong>Quantitation Workstation</strong>]</p></li>
<li><p>Vortex&nbsp;</p></li>
<li><p>Microcentrifuge&nbsp;</p></li>
<li><p>Thin-walled 0.5ml PCR tubes (Cat.# E4941 or Axygen Cat.# PCR-05-C). Make sure enough are available prior to beginning! You may need to autoclave a new set - these are available above the autoclave [<strong>Autoclave Workstation</strong>].</p></li>
<li><p>Nuclease-free water [Cabinet above water station]</p></li>
<li><p>Pre-label your tubes to speed things up!</p></li>
</ul>
</section>
</section>
<section id="feb-12---feb-18" class="level1">
<h1>Feb 12 - Feb 18</h1>
<p>JRE</p>
<p>This was the week we were supposed to start the extraction protocol, but got postponed.</p>
<p>Practiced Pipetting Water</p>
</section>
<section id="feb-20---feb-25" class="level1">
<h1>Feb 20 - Feb 25</h1>
</section>
<section id="feb-26---mar-3" class="level1">
<h1>Feb 26 - Mar 3</h1>
<p>February 29, 2024:</p>
<p>ELP and JRE:</p>
<ul>
<li>UV’d 1000𝜇l pipettes, 200𝜇l pipettes, 12 2ml lo-bind tubes, razor blades, 200 and 1000 𝜇l tips.</li>
<li>Set heatblock to 56C</li>
<li>ELP: made 10% bleach</li>
<li>ELP: made ice bath; put samples in ice bath</li>
<li>Wiped BSC down with 70% ethanol, water (twice), then 10% bleach</li>
</ul>
<p>Testing the 1000ml pipettes (ELP) (notetaker = FAB and JRE)</p>
<ul>
<li>They both look to be pipetting the same amount (marked with Sharpie)</li>
</ul>
<ol type="1">
<li>Pipet 200 𝜇l of fecal slurry into the microcentrifuge tube. You may need to cut the end of the pipet tip with a razor blade, if necessary, to make pipetting easier. Remember to vortex the sample prior to pipetting, to make sure you are getting a good mixture of buffer and feces.</li>
</ol>
<ul>
<li>Label tubes (completed by JRE at 5:13pm)</li>
<li>Adding 200 micro l water to control tube (completed by JRE at 5:16pm)</li>
<li>Vortex sample VOYT-18-017 and pipette 200 micro l into tube 03 (completed by ELP at 5:18pm)</li>
<li>Vortex sample VOYT-18-036 and pipette almost 200 micro l into tube 04 (completed by ELP at 5:19)</li>
<li>Vortex sample GIUNCHI-18-001 and pipette 200 micro l into tube 05 (completed by ELP at 5:20)</li>
<li>Vortex sample DIFIORE-18-025 and pipette 200 micro l into tube 06 (completed by ELP at 5:25)</li>
<li>Vortex sample DIFIORE-18-031 and pipette 200 micro l into tube 09 (completed by JRE at 5:28)</li>
<li>Vortex sample DIFIORE-18-032 and pipette 200 micro l into tube 11 (completed by JRE at 5:33)</li>
<li>Vortex sample VOYT-18-029 and pipette 200 micro l into tube 02 (completed by JRE at 5:35)</li>
<li>Vortex sample GOMEZ-18-190 and pipette 200 micro l into tube 01 (completed by FAB at 5:39)</li>
<li>Vortex sample VOYT-18-006 and pipette 200 micro l into tube 10 (completed by FAB at 5:44)</li>
<li>Vortex sample DIFIORE-18-026 and pipette 200 micro l into tube 07 (completed by FAB at 5:46)</li>
<li>Vortex sample DIFIORE-18-027 and pipette 200 micro l into tube 08 (completed by FAB at 5:48)</li>
</ul>
<ol start="2" type="1">
<li>Add 1.6 ml Buffer ASL to each stool sample. Vortex continuously for 1 min or until the stool sample is thoroughly homogenized. Note: It is very important to vortex the samples thoroughly. This helps ensure maximum DNA concentration in the final eluate.</li>
</ol>
<ul>
<li><p>Pipette 800 micro l of Buffer ASL to each tube twice, changing pipette tips each time (using 2 pipette tips per tube). (ELP completed control, tube 03, tube 04, and tube 05 at 5:53; JRE completed tube 06, 09, 11[slightly less buffer was added], and 02 at 6:03; FAB completed tube 01, 10, 07, and 08 at 6:08)</p></li>
<li><p>Vortex each of the tubes for 1 minute (2 at a time, 1 in each hand); double check that tubes are closed</p></li>
<li><p>Completed in shifts by ELP, JRE, and FAB at 6:17.</p></li>
</ul>
<ol start="3" type="1">
<li>Incubate samples at 56ºC in the Dry Bath for 30-60 min (60 minutes), then for an additional 30 min on Tube Rocker. Optional: further incubate in the Dry Bath at 23 degrees C overnight.</li>
</ol>
<ul>
<li>Started 56 degree C Dry Bath at 6:18 pm (completed at 7:22)
<ul>
<li>ELP, FAB, and JRE cleaned the BSC with ethanol, water, then bleach</li>
</ul></li>
<li>Started Tube Rocker at (completed at ), unknown, changes didn’t get pushed or committed
<ul>
<li>We got out of lab ~8PM JRE</li>
</ul></li>
<li>Left in overnight 23 degrees C Dry Bath
<ul>
<li>Dry Bath went down below 23 degrees C to 16.3 degrees C at 9:56 PM</li>
<li>Dry Bath was reset to 23 degrees C by FAB and was at temp at 10:02 PM</li>
</ul></li>
</ul>
<p>March 1, 2024</p>
<ul>
<li><p>Increase the temperature of the <span style="color:green">heating block</span> to 70 ºC</p>
<ul>
<li><p>Removed tubes from Dry Bath at 9:23 AM and placed in rack in BSC (at 23 degrees C), set Dry Bath to 70 degrees C (JRE)</p></li>
<li><p>ELP, FAB, and JRE mixed 10% bleach and wiped down BSC and microcentrifuge at 11:20 AM</p></li>
<li><p>ELP, FAB, and JRE UV’d the pipettes for 10min</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute to pellet stool particles (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li>ELP, FAB, and JRE started microcentrifuge at 20,000 rcf at 11:28 (took ~24s to reach speed) ended at 11:29</li>
</ul></li>
</ul>
<!-- -->
<ul>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p>
<ul>
<li>JRE labeled tubes at 11:27</li>
</ul></li>
<li><p>Pipet 1.4 ml of the supernatant into the new <span style="color:green">2 ml tubes</span> and discard the pellet</p>
<ul>
<li><p>ELP pipetted 700 micro l of supernatant 2 times (did not switch tips within tubes, 1 pipette tip was used per tube) into tubes 05, 03, 04, and 11. (completed at 11:37 am)</p></li>
<li><p>JRE pipetted 700 micro l of supernatant 2 times into tubes 10 and 02. The pellet was disturbed in tube 01. Tube 01 and the control were re-centrifuged at 20,000 rcf for 1 min. JRE continued with tube 06 then re-pipetted 01. (completed at 11:47 am)</p></li>
<li><p>FAB pipetted 700 micro l of supernatant 2 times into tubes 07, 09, 08, and CL. (completed at 11:52).</p></li>
<li><p>Pellets were tossed at 11:53, JRE UV’d an extra set of 1000 micro pipette tips</p></li>
</ul></li>
<li><p>Add 1 <span style="color:blue">InhibitEX Tablet</span> or 500 𝜇l of <span style="color:blue">liquid InhibitEX</span> (newer kits have liquid, older have tablets) to each sample</p>
<ul>
<li><p>ELP pipetted 500 micro l of liquid InhibitEX into tubes 05, 03, 04, and 11. A new pipette tip was used for each tube. (completed at 11:55)</p></li>
<li><p>JRE pipetted 500 micro l of liquid InhibitEX into tubes 10, 02, 06, 01 using a new pipette tip per tube. (completed at 11:58)</p></li>
<li><p>FAB pipetted 500 micro l of liquid InhibitEX into tubes 07, 09, 08, and CL using a new pipette tip per tube. (completed at 12:01).</p></li>
</ul></li>
<li><p>Vortex each sample immediately for 1 minute or until the <span style="color:blue">InhibitEX</span> is completely suspended</p>
<ul>
<li>JRE, ELP, and FAB switched off vortexing two tubes at a time (one in each hand) for 1 min. (completed at 12:08)</li>
</ul></li>
<li><p>Incubate suspension for 1 min at room temperature to allow inhibitors to adsorb to the <span style="color:blue">InhibitEX</span> matrix</p>
<ul>
<li>completed at 12:09.</li>
</ul></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p>
<ul>
<li>Completed by ELP at 12:04.</li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes to pellet stool particles and inhibitors bound to the <span style="color:blue">InhibitEX</span> matrix (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: for difficult samples centrifugation for 3 minutes may result in a pellet that is not sufficiently compact. Therefore it may be difficult to remove enough supernatant to transfer 600 𝜇l supernatant after the next centrifugation step. In these cases, we recommend to centrifuge for 6 minutes</li>
<li><strong>Note</strong>: When processing more than 12 samples, for this step and step 17 we recommend processing batches of no more than 12 samples each. This is because the pellets formed after centrifugation will break up quickly if the supernatant is not removed immediately</li>
<li>ELP and JRE complete microcentrifuging the 12 tubes at 20,000 rcf for 6 min at 12:19.</li>
</ul></li>
<li><p>Immediately after the <span style="color:green">microcentrifuge</span> stops, pipet all of the supernatant into the clean <span style="color:green">2 ml tubes</span> and discard the pellet (Transferring small quantities of pelleted material from the previous step will not affect the procedure)</p>
<ul>
<li><p>Pipette was set to 900 micro l.</p></li>
<li><p>ELP pipetted 900 micro l of supernatant twice (switching pipette tips between tubes) into tubes 03, 04, 05, and 11. Completed at 12:25.</p></li>
<li><p>JRE pipetted 900 micro l of supernatant twice (switching pipette tips between tubes) into tubes 10, 02, 06, 01. Completed at 12:30.</p></li>
<li><p>FAB pipetted 900 micro l of supernatant twice (switching pipette tips between tubes) into tubes 07, 08, 09, and CL. Completed at 12:37</p></li>
</ul></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p>
<ul>
<li>completed by ELP at 12:16.</li>
</ul></li>
<li><p>Pipet 25 𝜇l of <span style="color:blue">proteinase K</span> into the newly labeled tubes</p>
<ul>
<li><p>JRE UV’d a box of 10 micro pipette tips for 10 min. Completed at 12:27. (this was necessary)</p></li>
<li><p>ELP pipetted 25 micro l of proteinase K (switching pipette tips between tubes) into the newly labeled empty 2 ml tubes 03, 04, 05, and 11. Completed at 12:40.</p></li>
<li><p>JRE pipetted 25 micro l of proteinase K (switching pipette tips between tubes) into the newly labeled empty 2ml tubes 10, 02, 06, 01. Completed at 12:43</p></li>
<li><p>FAB pipetted 25 micro l of proteinase K (switching pipette tips between tubes) into the newly labeled and empty 2 ml tubes 07, 08, 09, and CL. Completed at 12:46.</p></li>
</ul></li>
<li><p>Pipet 600 𝜇l supernatant from step 18 into the tubes containing the <span style="color:blue">proteinase K</span></p>
<ul>
<li><p>ELP pipetted 600 micro l of supernatant into the new proteinase K tubes labeled 03, 04, 05, and 11. Completed at 12:49.</p></li>
<li><p>JRE pipetted 600 micro l of supernatant into the new proteinase K tubes labeled 10, 02, 06, 01. Completed at 12:52.</p></li>
<li><p>FAB pipetted 600 micro l of supernatant into the new proteinase K tubes labeled 07, 08, 09, and CL. Completed at 12:55).</p></li>
<li><p>A pink post-it note was added to the tray with the leftover supernatant tubes and set to the side to distinguish between those and our active sample tubes in a separate tray.</p></li>
</ul></li>
<li><p>Add 600 𝜇l <span style="color:blue">Buffer AL</span></p>
<ul>
<li><strong>Note</strong>: do <strong>NOT</strong> add <span style="color:blue">proteinase K</span> directly to <span style="color:blue">Buffer AL</span>. It is essential that the sample and <span style="color:blue">Buffer AL</span> are thoroughly mixed to form a homogeneous solution</li>
<li>ELP pipetted 600 micro l of Buffer AL (switching pipette tips between tubes) into the active sample tubes 03, 04, 05, and 11. Completed at 12:58.</li>
<li>JRE pipetted 600 micro l of Buffer AL (switching pipette tips between tubes) into the active sample tubes 10, 02, 06, and 01. (Had to switch to using Buffer AL from the QIAmp DNA Blood Kit for tubes 10 and 02) Completed at 1:02.</li>
<li>FAB pipetted 60 micro l of Buffer AL (from Blood Kit) into the active sample tubes 07, 08, 09, and CL. Completed at 1:06.</li>
</ul></li>
<li><p>Vortex tubes for 15 seconds</p>
<ul>
<li>ELP Vortexed all 12 tubes for 15 sec (2 at a time). Completed at 1:08.</li>
</ul></li>
<li><p>Incubate the tubes in the <span style="color:green">heating block</span> at 70°C for 60 min</p>
<ul>
<li>You may want to briefly centrifuge the tubes using the small <span style="color:green">benchtop centrifuge</span> at the <strong>Quantification Station</strong> to remove drops from the inside of the tube lid (make sure that the centrifuge is <strong>balanced</strong>)</li>
<li>JRE started 60 min incubation at 1:10pm.</li>
<li>Leftover supernatant tray was placed in the -20 fridge by ELP at 1:11.</li>
<li>ELP took all 12 lysates out of Heat Block at 2:11.</li>
<li>ELP benchtop centrifuged all 12 lysates for 5 seconds.</li>
</ul></li>
<li><p>Add 600 𝜇l of <span style="color:green">96–100 % ethanol</span> to the lysate</p>
<ul>
<li><p>FAB had an exam, not present for this part of protocol.</p></li>
<li><p>ELP aliquotted 7.2mL of 96-100% ethanol into 2 LoBind Tubes (refilled in-between ELP and JRE pipetting ethanol). Completed at 2:17.</p></li>
<li><p>ELP pipetted 600 micro l of 96-100% ethanol (switching pipette tips between tubes) to lysates 03, 04, 05, 11, 07, 09. Completed at 2:16.</p></li>
<li><p>JRE pipetted 600 micro l of 96-100% ethanol (switching pipette tips between tubes) to lysates CL, 08, 02, 06, 01, and 10. Completed at 2:21.</p></li>
</ul></li>
<li><p>Vortex briefly</p>
<ul>
<li>You may want to briefly centrifuge the tubes using the small <span style="color:green">benchtop centrifuge</span> at the <strong>Quantification Station</strong> to remove drops from the inside of the tube lid (make sure that the centrifuge is <strong>balanced</strong>)</li>
<li>ELP vortexed all 12 tubes for 15 sec (2 at a time). Centrifuged 6 tubes at a time for ~5s. Completed at 2:28.</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">QIAamp spin columns</span> and <span style="color:blue">2 ml collection tubes</span></p>
<ul>
<li>Completed by JRE at 2:29</li>
</ul></li>
<li><p>Carefully apply 600 𝜇l of the lysate to the <span style="color:blue">QIAamp spin columns</span> without moistening the rim and close the cap</p>
<ul>
<li><p>ELP pipetted 600 micro l of lysate (switching pipette tips between tubes) to QIAamp spin columns CL, 01, 02, 03, 04, 05. Completed at 2:33.</p></li>
<li><p>JRE pipetted 600 micro l of lysate (switching pipette tips between tubes) to QIAamp spin columns 06, 07. 08, 09, 10, 11. Completed at 2:38.</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li>Completed by ELP at 2:41 (lid was not on, made a scary noise).</li>
</ul></li>
<li><p>Carefully pull the <span style="color:blue">QIAamp spin columns</span> out of the <span style="color:blue">collection tubes</span> and discard the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin columns</span> are empty</li>
<li>Sample 10 and 08 did not completely filter out. Some of Sample 10 spilled out onto centrifuge (wiped with kimwipe and Bleach). ELP redid and completed centrifuge step for these samples at 2:47</li>
</ul></li>
<li><p>Replace the <span style="color:blue">QIAamp spin columns</span> back into the now empty <span style="color:blue">collection tubes</span></p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span>, apply a second aliquot of 600 𝜇l lysate and close the cap</p>
<ul>
<li><p>ELP emptied collection tubes CL, 01, 02, 03, 04, 05 and pipetted 600 micro l of lysate (switching pipette tips between tubes) to QIAamp spin columns. Completed at 2:53.</p></li>
<li><p>JRE emptied collection tubes 06, 07, 08, 09, 10, 11 and pipetted 600 micro l of lysate (switching pipette tips between tubes) to QIAamp spin columns. Completed at 2:58.</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li>Completed by JRE at 3:01</li>
</ul></li>
<li><p>Carefully pull the <span style="color:blue">QIAamp spin columns</span> out of the <span style="color:blue">collection tubes</span> and discard the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
</ul></li>
<li><p>Replace the <span style="color:blue">QIAamp spin columns</span> back into the now empty <span style="color:blue">collection tubes</span></p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span>, apply a third aliquot of 600 𝜇l lysate and close the cap</p>
<ul>
<li><p>ELP emptied collection tubes CL, 01, 02, 03, 04, 05 (wiping the rim of each tube with 1 kimwipe) and pipetted final 600 micro l of lysate (switching pipette tips between tubes) to QIAamp spin columns. Discarded old tubes Completed at 3:10</p></li>
<li><p>JRE emptied collection tubes 06, 07, 08, 09, 10, 11 (wiping the rim of each tube with 1 kimwipe) and pipetted final 600 micro l of lysate (switching pipette tips between tubes) to QIAamp spin columns. Discard old tubes Completed at 3:15</p></li>
<li><p>Note, final micro l of lysate for all tubes not quite 600 micro l.</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><p>Completed by ELP at 3:16.</p></li>
<li><p>Sample 07 had to be respun. Completed by ELP and JRE at 3:20</p></li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span> and add 500 𝜇l of <span style="color:blue">Buffer AW1</span> and close the cap</p>
<ul>
<li><p>JRE disposed and replaced each collection tube, then pipetted 500 micro l of Buffer AW1 to spin columns 06, 07, 08, 09, 10, 11. Completed at 3:25.</p></li>
<li><p>ELP disposed and replaced each collection tube, then pipetted 500 micro l of Buffer AW1 to spin columns CL, 01, 02, 03, 04, 05. Completed at 3:29.</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span> and add 500 𝜇l of <span style="color:blue">Buffer AW2</span> and close the cap</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><p>ELP started at 4:41</p>
<ul>
<li><strong>Note</strong>: residual <span style="color:blue">Buffer AW2</span> in the eluate may cause problems in downstream applications. Some centrifuge rotors may vibrate upon deceleration, resulting in the flow-through, which contains <span style="color:blue">Buffer AW2</span>, contacting the <span style="color:blue">QIAamp spin column</span>. Removing the <span style="color:blue">QIAamp spin column</span> and <span style="color:blue">collection tube</span> from the rotor may also cause flow-through to come into contact with the <span style="color:blue">QIAamp spin column</span></li>
</ul></li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: this step is optional but recommended help eliminate the chance of <span style="color:blue">Buffer AW2</span> carryover
<ul>
<li>ELP started at 3:52</li>
</ul></li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate (the filtrate should be minimal)</p></li>
<li><p>Place <span style="color:blue">Buffer AE</span> in the waterbath at 70 ºC to incubate for 10 minutes</p>
<ul>
<li>ELP started at 3:33</li>
</ul></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span> (These are the tubes going into the freezer, so labeling is extra important!)</p></li>
<li><p>Transfer the <span style="color:blue">QIAamp spin columns</span> into the new set of <span style="color:green">2 ml tubes</span></p></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin columns</span>, pipet 100 𝜇l of <span style="color:blue">Buffer AE</span> directly onto the QIAamp membrane (do not touch the membrane with the pipette tip), and close the cap</p>
<ul>
<li>ELP started at 4:12.</li>
</ul></li>
<li><p>Incubate the tubes at room temperature for 25 minutes</p>
<ul>
<li>ELP started at 4:15</li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute to elute the DNA (you will need to use the largest centrifuge at the <strong>Centrifuge Station</strong> as this is the only one that goes this fast)</p>
<ul>
<li>ELP and JRE completed at 4:48.</li>
</ul></li>
</ul>
<!-- -->
<ul>
<li><p>Pull the filter portion of the tube up and cap the tube (you can either discard it or save them if you want to try and get more DNA out of them)</p>
<ul>
<li>ELP completed at 4:56.</li>
</ul></li>
<li><p>The liquid in the tube contains the DNA from the samples and should be stored in the <span style="color:green">-20 °C freezer</span> when not in use! (Be sure to record where in the freezer you’ve placed them, and make sure that they are all together in a single, clearly labeled container/rack)</p></li>
<li><p>The next step is to quantify the DNA in the samples (this is done by measuring the dsDNA concentration using the <span style="color:green">Quantus</span> and the sample A260/A280 using the <span style="color:green">Nanodrop</span>, both are located at the <strong>Extraction Workstation</strong>). The results should be recorded in your lab notebook</p>
<ul>
<li><p>JRE labeled 0.5ml tubes and began UV’ing 10micro and 200 micro l pipette tips and pipettes at 4:50.</p>
<p>1. If necessary, prepare <strong>1X TE Buffer</strong>: to do this, you may need to dilute <strong>20X TE Buffer</strong> 20-fold with <strong>nuclease-free water</strong>. We do also have 1X TE BUffer available in the cabinet above the Centrifuge Workstation; there may also be pre-prepared 1X TE Buffer in the Underbench Fridge.</p></li>
<li><p>Done prior to starting.</p>
<ul>
<li><p>2. Prepare Working Solution: Dilute the <strong>QuantiFluor® dsDNA Dye</strong> 1:400 in <strong>1X TE buffer</strong>, and mix. Working Solution may be available in the Underbench Fridge.</p>
<ul>
<li><p>Done prior to starting.</p></li>
<li><p>Ran out of Working solution at 11th tube, ELP and JRE made ~1ml (3 parts dye to 1,197 TEx1) Working solution following protocol at 5:34.</p>
<p>3. Prepare Blank sample: Mix 2 μl of <strong>1X TE buffer</strong> with 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> Working Solution in an empty <strong>0.5 ml PCR tube</strong>. <strong>Vortex</strong> well and protect tube from light (we do this by covering the tube with aluminum foil).</p></li>
<li><p>JRE pipetted 2 micro l of 1X TE buffer into 0.5ml tube at 5:10</p></li>
<li><p>ELP pipetted 200 micro l of Working solution at 5:24.</p>
<p>4. Prepare Standard: Add 2 μl of the provided <strong>DNA Standard</strong> (100 ng/μl) to 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> working solution in an empty <strong>0.5 ml PCR tube</strong>. <strong>Vortex</strong> well and protect tube from light.</p></li>
<li><p>JRE pipetted 2 microl l of DNA Standard into 0.5 tube at 5:11</p></li>
<li><p>ELP pipetted 200 micro l of Working solution at 5:25.</p>
<p>5. Prepare Samples to be quantified: Add 1 or 2 μl of unknown samples to 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> working solution in <strong>0.5 ml PCR tubes</strong>. <strong>Vortex</strong> well, and protect tube from light. You will want to pipet the sample up and down multiple times to make sure the sample is well-blended.</p></li>
<li><p>JRE pipetted 2 micro l from samples CL, 01, 02, 03, 04. Completed at 5:16.</p></li>
<li><p>ELP pipetted 200 micro l of Working solution to samples CL, 01, 02, 03, 04, 05. Completed at 5:28.</p></li>
<li><p>ELP pipetted 2 micro l from samples 05, 06, 07, 08, 09, 10, 11. Completed at 5:22.</p></li>
<li><p>JRE pipetted 200 micro l of Working solution to samples 06, 07, 08, 09, 10, 11</p></li>
<li><p>Completed (after remaking ~1ml Working solution) at 5:44.</p>
<p>6. Incubate the prepared samples at room temperature for 5 minutes, protected from light.</p></li>
<li><p>Finished incubating at 5:49.</p>
<p>7. Select the dsDNA protocol on the <strong>Quantus Fluorometer</strong>.</p></li>
<li><p>Completed by ELP at 5:45.</p>
<p>8. Prior to running samples on the Quantus Fluorometer, you should blend them well by <strong>vortexing</strong> briefly and then spinning them down briefly using the <strong>microcentrifuge</strong>. You should also wipe down the sides of the tubes with <strong>kimwipes</strong> to make sure the sides are clear from smudges.&nbsp;</p></li>
<li><p>ELP vortexed and centrifuged each sample.</p>
<p>9. If needed (you will need it!), calibrate the <strong>Quantus Fluorometer</strong> by reading the blank (Step 3; this should read as 0 ng/μl) and standard (Step 4; this should read as 100 ng/μl) samples in the Calibration screen, then select “Save”.&nbsp;</p>
<ul>
<li><strong>Note</strong>: a few ng/μl away from what it should be is ok… but large deviations need to be troubleshooted!</li>
</ul></li>
<li><p>Calibration complete at 5:54.</p>
<p>10. Enter the volume of the unknown sample (1–2 μl used in Step 5) and desired concentration units (in our case, ng/μl).</p></li>
<li><p>Already set prior to starting</p>
<p>11. Pipette to mix each sample at least 10 times before putting the desired amount in the working solution.&nbsp;</p></li>
<li><p>Done</p>
<p>12. One sample at a time, mix by vortexing, briefly spin down and wipe the outside of the tube with a kimwipe. Measure fluorescence of the unknown sample and record the final sample concentration results.</p></li>
</ul>
<!-- -->
<ul>
<li><strong>Note</strong>: DNA yield is typically 15–60 𝜇g but, depending on the individual stool sample and the way it was stored, may range from 5 to 100 𝜇g. DNA concentration is typically 75–300 ng/𝜇l
<ul>
<li>Blank concentration: 0</li>
<li>Standard concentration: 101</li>
</ul></li>
</ul></li>
</ul></li>
</ul></li>
</ul>
<table class="table">
<colgroup>
<col style="width: 14%">
<col style="width: 13%">
<col style="width: 7%">
<col style="width: 17%">
<col style="width: 12%">
<col style="width: 32%">
</colgroup>
<thead>
<tr class="header">
<th>Sample ID</th>
<th>Taxon</th>
<th>Label</th>
<th>Final Concentration (ng/𝜇l)</th>
<th><strong>A260/A280 value</strong></th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Control (sterile water)</td>
<td>NA</td>
<td>AGI24-DBCL</td>
<td>0.0258 -Q 8-N</td>
<td>5.7</td>
<td></td>
</tr>
<tr class="even">
<td>GOMEZ-18-190</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB01</td>
<td>1.46 - Q 8.6-N</td>
<td>5.66</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-029</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB02</td>
<td>0.234 - Q 9.8-N</td>
<td>4.31</td>
<td></td>
</tr>
<tr class="even">
<td>VOYT-18-017</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB03</td>
<td>0.395 - Q 7.1 -N</td>
<td>8.38</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-036</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB04</td>
<td>0.195- Q 8.3-N</td>
<td>5.92</td>
<td>Maybe not the entire 200𝜇l</td>
</tr>
<tr class="even">
<td>GIUNCHI-18-001</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB05</td>
<td>0.595 - Q 10.2-N</td>
<td>3.97</td>
<td></td>
</tr>
<tr class="odd">
<td>DIFIORE-18-025</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB06</td>
<td>0.202 - Q 10.3-N</td>
<td>4.15</td>
<td></td>
</tr>
<tr class="even">
<td>DIFIORE-18-026</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB07</td>
<td>1.09 - Q 11.2-N</td>
<td>4.22</td>
<td></td>
</tr>
<tr class="odd">
<td>DIFIORE-18-027</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB08</td>
<td>0.145 - Q 8.5-N</td>
<td>4.17</td>
<td></td>
</tr>
<tr class="even">
<td>DIFIORE-18-031</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB09</td>
<td>0.151-Q 8.4-N</td>
<td>6.94</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-006</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB10</td>
<td>1.82-Q 15.4-N</td>
<td>3.03</td>
<td>Pipetted most of 200 then a tiny bit extra some chunks</td>
</tr>
<tr class="even">
<td>DIFIORE-18-032</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB11</td>
<td>0.315-Q 14.13-N</td>
<td>6.9</td>
<td>Slightly less buffer was added</td>
</tr>
</tbody>
</table>
<p>JRE and ELP finished (sad) at 6:00.</p>
</section>
<section id="mar-4---mar-10" class="level1">
<h1>Mar 4 - Mar 10</h1>
<p>March, 4, 2024</p>
<p>Written prior to there being a protocol on the website.</p>
<ul>
<li>JRE selected nucleic acids option on the</li>
</ul>
<ol type="1">
<li>Clean pedestal w kimwipes and nuclease free water</li>
<li>Blank w 2 micro l of nuclease free water or storage medium (Buffer ATE)
<ul>
<li>JRE blanked with nuclease free water, not Buffer ATE.</li>
</ul></li>
<li>Between each sample, clean w kim wipe and nuclease free water</li>
<li>2 micro l of sample for each run</li>
</ol>
<p>JRE, FAB, and ELP finished at 12:12</p>
<section id="march-7-2024" class="level2">
<h2 class="anchored" data-anchor-id="march-7-2024">March 7, 2024</h2>
<p>Second batch of extractions!</p>
<p>3:59, JRE and FAB created 10% bleach, finished cleaning biosafety cabinet, and UV’d pipettes and tips for 15 minutes.</p>
<ul>
<li><p>FAB warmed up frozen supernatant with her hands started at 4:07 and completed at 4:12.</p></li>
<li><p>FAB vortexed each sample for 10 seconds. completed at 4:23</p></li>
<li><p>*Continuing protocol from step 8 onwards</p></li>
</ul>
<ol type="1">
<li><p>Label a new set of <span style="color:green">2 ml tubes</span></p>
<ul>
<li>JRE completed at 4:22</li>
</ul></li>
<li><p>Pipet 25 𝜇l of <span style="color:blue">proteinase K</span> into the newly labeled tubes</p>
<ul>
<li><p>JRE completed pipetting 25 micro l of proteinase k into new tubes CL, 01, 02, 03, 04, 05 at 4:30</p></li>
<li><p>FAB completed pipetting 25 micro l of proteinase k into new tubes 06, 07, 08, 09, 10, 11 at 4:33</p></li>
</ul></li>
<li><p>Pipet 600 𝜇l leftover supernatant from step 18 into the tubes containing the <span style="color:blue">proteinase K</span></p>
<ul>
<li><p>FAB completed pipetting 600 micro l of supernatant into new tubes 06, 07, 08, 09, 10, 11 at 4:39</p>
<ul>
<li>ELP and FAB vortexed samples for 1 minute, started 4:40, completed 4:44.</li>
</ul>
<p>JRE completed pipetting 600 micro l of supernatant into new tubes CL, 01, 02, 03, 04, 05 at. Sample 5 spilled a bit, had to restart, (might be slightly under since it was the final bit of supernatant) added proteinase k to supernatant leftover/old tube. Completed at at 4:48.</p>
<ul>
<li><p>ELP and JRE vortexed samples for 1 minute starting at 4:49, completed at 5:52.</p></li>
<li><p>The QIAamp Fast DNA Stool Mini Kit is expired, by well over 12 months. The DNAeasy PowerLyzer PowerSoil Kit is not expired.</p></li>
</ul></li>
</ul></li>
<li><p>Add 600 𝜇l <span style="color:blue">Buffer AL</span></p>
<ul>
<li><strong>Note</strong>: do <strong>NOT</strong> add <span style="color:blue">proteinase K</span> directly to <span style="color:blue">Buffer AL</span>. It is essential that the sample and <span style="color:blue">Buffer AL</span> are thoroughly mixed to form a homogeneous solution</li>
<li>JRE added 600 micro l of buffer AL to tubes CL, 01,02,03. Completed at 4:59.</li>
<li>ELP added 600 micro l of buffer AL to tubes 04,05,06,07. Completed at 5:01.</li>
<li>(ran low on buffer AL, switched to different DNeasy blood and tissue kit Buffer AL) FAB added 600 micro l of buffer AL to tubes 08,09,10,11. Completed at 5:04.</li>
</ul></li>
<li><p>Vortex tubes for 15 seconds (We are trying 1 minute instead)</p>
<ul>
<li>JRE vortexed all samples for 1 min. Completed at 5:13.</li>
</ul></li>
<li><p>Incubate the tubes in the <span style="color:green">heating block</span> at 70°C for 60 min</p>
<p>You may want to briefly centrifuge the tubes using the small <span style="color:green">benchtop centrifuge</span> at the <strong>Quantification Station</strong> to remove drops from the inside of the tube lid (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li>Started incubation at 5:14. Completed at 6:14.</li>
</ul></li>
<li><p>Add 600 𝜇l of <span style="color:green">96–100 % ethanol</span> to the lysate</p>
<ul>
<li><p>ELP completed benchtop centrifuging all samples at 6:18.</p></li>
<li><p>JRE added 600 micro l of 100% ethanol to tubes CL, 01, 02, 03. Completed at 6:21.</p></li>
<li><p>ELP added 600 micro l of 100% ethanol to tubes 04, 05, 06, 07. Completed at 6:22</p></li>
<li><p>FAB added 600 micro l of 100% ethanol to tubes 08, 09, 10, 11. Completed at 6:25</p></li>
</ul></li>
<li><p>Vortex briefly</p>
<ul>
<li>ELP vortexed all samples for 1 min, completed at 6:31</li>
<li>JRE benchtop centrifuged all samples. Completed at 6:34</li>
<li>You may want to briefly centrifuge the tubes using the small <span style="color:green">benchtop centrifuge</span> at the <strong>Quantification Station</strong> to remove drops from the inside of the tube lid (make sure that the centrifuge is <strong>balanced</strong>)</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">QIAamp spin columns</span> and <span style="color:blue">2 ml collection tubes</span></p>
<ul>
<li><p>JRE labeled all spin columns and 2ml tubes, completed at 6:26.</p></li>
<li><p>Note: Perhaps the protocol didn’t work as well the first time because the samples weren’t agitated enough? Per the QIAamp protocol we could have heated the suspended samples at 70C-95C for pathogen analysis (plant cell walls are tougher to break).</p></li>
</ul></li>
<li><p>Carefully apply 600 𝜇l of the lysate to the <span style="color:blue">QIAamp spin columns</span> without moistening the rim and close the cap</p>
<ul>
<li><p>JRE applied 600 micro l of lysate from samples CL, 01, 02, 03 to their labeled tubes. Completed at 6:37.</p></li>
<li><p>ELP applied 600 micro l of lysate from samples 04, 05, 06, 07 to their labeled tubes. Completed at 6:40</p></li>
<li><p>FAB applied 600 micro l of lysate from samples 08, 09, 10, 11 to their labeled tubes. Completed at 6:44</p>
<ul>
<li>FAB spilled a slight amount of sample 08 lysate at 6:41, cleaned with bleach and kimwipes.</li>
</ul></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li>ELP completed at 6:48.</li>
</ul></li>
<li><p>Carefully pull the <span style="color:blue">QIAamp spin columns</span> out of the <span style="color:blue">collection tubes</span> and discard the filtrate</p>
<ul>
<li>JRE discarded filtrate and kimwiped all tubes. Completed at 6:55.</li>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin columns</span> are empty</li>
</ul></li>
<li><p>Replace the <span style="color:blue">QIAamp spin columns</span> back into the now empty <span style="color:blue">collection tubes</span></p>
<ul>
<li>Completed by JRE in previous step at 6:55.</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span>, apply a second aliquot of 600 𝜇l lysate and close the cap</p>
<ul>
<li><p>JRE added 600 micro l of lysate to tubes CL, 01, 02, 03. Completed at 6:57.</p></li>
<li><p>ELP added 600 micro l of lysate to tubes 04, 05, 06, 07. Completed at 7:00.</p></li>
<li><p>FAB added 600 micro l of lysate to tubes 08, 09, 10, 11. Completed at 7:03</p>
<ul>
<li>Pipetted up bubbles for lysate sample 10 (tiny bit of lysate likely got left behind)</li>
</ul></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li>Completed by ELP and FAB at 7:07</li>
</ul></li>
<li><p>Carefully pull the <span style="color:blue">QIAamp spin columns</span> out of the <span style="color:blue">collection tubes</span> and discard the filtrate</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
<li>FAB discarded filtrate and kimwiped all tubes. Completed at 7:13.</li>
</ul></li>
<li><p>Replace the <span style="color:blue">QIAamp spin columns</span> back into the now empty <span style="color:blue">collection tubes</span></p>
<ul>
<li>Completed by FAB in previous step at 7:13.</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span>, apply a third aliquot of 600 𝜇l lysate and close the cap</p>
<ul>
<li><p>FAB added final 600 micro l of lysate to tubes 08, 09, 10, 11. Completed at 7:17.</p></li>
<li><p>ELP added final 600 micro l of lysate to tubes 04, 05, 06, 07. Completed at 7:19.</p></li>
<li><p>JRE added final 600 micro l of lysate to tubes CL, 01, 02, 03. Completed at 7:23</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li>ELP completed micro centrifuging at 7:25.</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p>
<ul>
<li>(used lid-less 2ml tubes, did not label)</li>
</ul></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p>
<ul>
<li><p>Completed by ELP at 7:29.</p>
<ul>
<li><strong>Note</strong>: if the lysate has not completely passed through the column after centrifugation, centrifuge again until the <span style="color:blue">QIAamp spin column</span> is empty</li>
</ul></li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span> and add 500 𝜇l of <span style="color:blue">Buffer AW1</span> and close the cap</p>
<ul>
<li><p>JRE added 500 micro l of Buffer AW1 to tubes CL, 01, 02, 03. Completed at 7:33.</p></li>
<li><p>ELP added 500 micro l of Buffer AW1 to tubes 04, 05, 06, 07. Completed at 7:35.</p></li>
<li><p>FAB added 500 micro l of Buffer AW1 to tubes 08, 09,10, 11. Completed at 7:38</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li>JRE microcentrifuged tubes for 1 minute. Completed at 7:42. Tubes 9 and 11 did not finish filtering, centrifuged for an additional minute. Completed at 7:45.</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p>
<ul>
<li>(used lid-less 2ml tubes, did not label)</li>
</ul></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p>
<ul>
<li>Completed by ELP at 7:46.</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin column</span> and add 500 𝜇l of <span style="color:blue">Buffer AW2</span> and close the cap</p>
<ul>
<li><p>JRE added 500 micro l of Buffer AW2 to tubes CL, 01, 02, 03. Completed at 7:49.</p></li>
<li><p>ELP added 500 micro l of Buffer AW2 to tubes 04, 05, 06, 07. Completed at 7:53.</p>
<ul>
<li>Pipette came apart, JRE UV’d a new 1000 micro l pipette for 10 min. Completed at</li>
</ul></li>
<li><p>FAB added 500 micro l of Buffer AW2 to tubes 08, 09, 10, 11. Completed at 7:55</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 3 minutes (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: residual <span style="color:blue">Buffer AW2</span> in the eluate may cause problems in downstream applications. Some centrifuge rotors may vibrate upon deceleration, resulting in the flow-through, which contains <span style="color:blue">Buffer AW2</span>, contacting the <span style="color:blue">QIAamp spin column</span>. Removing the <span style="color:blue">QIAamp spin column</span> and <span style="color:blue">collection tube</span> from the rotor may also cause flow-through to come into contact with the <span style="color:blue">QIAamp spin column</span></li>
<li>Completed by FAB and ELP at 8:02.</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p>
<ul>
<li>Completed by JRE at 7:58</li>
</ul></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate</p>
<ul>
<li>Completed by ELP and JRE at 8:04.</li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>)</p>
<ul>
<li><strong>Note</strong>: this step is optional but recommended help eliminate the chance of <span style="color:blue">Buffer AW2</span> carryover</li>
<li>Completed by ELP at 8:07</li>
</ul></li>
<li><p>Label the lid of a new set of <span style="color:blue">2 ml collection tubes</span></p>
<ul>
<li>(used lid-less tubes, did not label)</li>
</ul></li>
<li><p>Place the <span style="color:blue">QIAamp spin columns</span> into the new set of labeled <span style="color:blue">2 ml collection tubes</span> and discard the tubes containing the filtrate (the filtrate should be minimal)</p>
<ul>
<li>Completed by ELP at 8:09.</li>
</ul></li>
<li><p>Place <span style="color:blue">Buffer AE</span> in the waterbath at 70 ºC to incubate for 10 minutes</p>
<ul>
<li>JRE put in waterbath at 7:56, kept in bath until 8:14.</li>
</ul></li>
<li><p>Label a new set of <span style="color:green">2 ml tubes</span> (These are the tubes going into the freezer, so labeling is extra important!)</p>
<ul>
<li>(used 1.5 ml tubes) JRE labeled at 8:10.</li>
</ul></li>
<li><p>Transfer the <span style="color:blue">QIAamp spin columns</span> into the new set of <span style="color:green">2 ml tubes</span></p>
<ul>
<li>Completed by ELP at 8:13</li>
</ul></li>
<li><p>Carefully open the <span style="color:blue">QIAamp spin columns</span>, pipet 100 𝜇l of <span style="color:blue">Buffer AE</span> directly onto the QIAamp membrane (do not touch the membrane with the pipette tip), and close the cap</p>
<ul>
<li>NOTE: Online protocol calls for Buffer AE, kit came only with Buffer ATE (and kit protocol uses ATE at this step)</li>
</ul>
<!-- -->
<ul>
<li><p>JRE added 100 micro l of Buffer ATE to tubes CL, 01, 02, 03. Completed at 8:17.</p></li>
<li><p>ELP added 100 micro l of Buffer ATE to tubes 04, 05, 06, 07. Completed at 8:19.</p></li>
<li><p>FAB added 100 micro l of Buffer ATE to tubes 08, 09, 10,11. Completed at 8:22.</p></li>
</ul></li>
<li><p>Incubate the tubes at room temperature for 25 minutes</p>
<ul>
<li><p>Incubated in dry bath at 20 c.&nbsp;Started at 8:22. Completed at 8:47</p></li>
<li><p>FAB used the restroom (not in the lab)</p></li>
</ul></li>
<li><p>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 20,000 rcf (not rpm) for 1 minute to elute the DNA (you will need to use the largest centrifuge at the <strong>Centrifuge Station</strong> as this is the only one that goes this fast)</p>
<ul>
<li><p>ELP placed tubes (spin column “loosely” in 1.5ml tube) in the centrifuge in pairs with the lids facing outward.</p></li>
<li><p>Completed centrifuging at 8:55.</p></li>
</ul></li>
<li><p>Pull the filter portion of the tube up and cap the tube (you can either discard it or save them if you want to try and get more DNA out of them)</p>
<ul>
<li><p>JRE labeled tubes to save spin columns “Spin Column 2” at 8:28</p></li>
<li><p>ELP removed spin columns from 1.5 ml tubes and put them in lid-less 2 ml tubes. Completed at 8:58.</p></li>
</ul></li>
<li><p>The liquid in the tube contains the DNA from the samples and should be stored in the <span style="color:green">-20 °C freezer</span> when not in use! (Be sure to record where in the freezer you’ve placed them, and make sure that they are all together in a single, clearly labeled container/rack)</p>
<ul>
<li>ELP and JRE placed spin columns and 1.5 ml tubes in freezer at 9:01.</li>
</ul></li>
<li><p>The next step is to quantify the DNA in the samples (this is done by measuring the dsDNA concentration using the <span style="color:green">Quantus</span> and the sample A260/A280 using the <span style="color:green">Nanodrop</span>, both are located at the <strong>Extraction Workstation</strong>). The results should be recorded in your lab notebook</p>
<ul>
<li><strong>Note</strong>: DNA yield is typically 15–60 𝜇g but, depending on the individual stool sample and the way it was stored, may range from 5 to 100 𝜇g. DNA concentration is typically 75–300 ng/𝜇l</li>
</ul></li>
</ol>
<ul>
<li>ELP cleaned the BSC with ethanol, water, and bleach at 8:30.</li>
</ul>
</section>
<section id="march-8-2024" class="level2">
<h2 class="anchored" data-anchor-id="march-8-2024">March 8, 2024</h2>
<ul>
<li><p>1. If necessary, prepare <strong>1X TE Buffer</strong>: to do this, you may need to dilute <strong>20X TE Buffer</strong> 20-fold with <strong>nuclease-free water</strong>. We do also have 1X TE BUffer available in the cabinet above the Centrifuge Workstation; there may also be pre-prepared 1X TE Buffer in the Underbench Fridge.</p></li>
<li><p>JRE made and aliquotted 1X TE Buffer at 12:14.</p>
<ul>
<li><p>2. Prepare Working Solution: Dilute the <strong>QuantiFluor® dsDNA Dye</strong> 1:400 in <strong>1X TE buffer</strong>, and mix. Working Solution may be available in the Underbench Fridge.</p>
<ul>
<li><p>ELP and JRE made 3.6 ml (3 parts dye to 1,197 TEx1 three times) Working solution following protocol at 12:23.</p></li>
<li><p>JRE vortexed working solution at 12:31.</p></li>
<li><p>JRE added 200 micro l of working solution into 14 autoclaved 0.5 ml tubes. Completed at 12:36.</p>
<p>3. Prepare Blank sample: Mix 2 μl of <strong>1X TE buffer</strong> with 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> Working Solution in an empty <strong>0.5 ml PCR tube</strong>. <strong>Vortex</strong> well and protect tube from light (we do this by covering the tube with aluminum foil).</p></li>
<li><p>ELP pipetted 2 micro l of 1X TE buffer into 0.5ml tube at 12:37</p>
<p>4. Prepare Standard: Add 2 μl of the provided <strong>DNA Standard</strong> (100 ng/μl) to 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> working solution in an empty <strong>0.5 ml PCR tube</strong>. <strong>Vortex</strong> well and protect tube from light.</p></li>
<li><p>ELP pipetted 2 microl l of DNA Standard into 0.5 tube at 12:39</p>
<p>5. Prepare Samples to be quantified: Add 1 or 2 μl of unknown samples to 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> working solution in <strong>0.5 ml PCR tubes</strong>. <strong>Vortex</strong> well, and protect tube from light. You will want to pipet the sample up and down multiple times to make sure the sample is well-blended.</p></li>
<li><p>JRE pipetted 2 micro l from samples CL, 01, 02, 03. Completed at 12:44.</p></li>
<li><p>ELP pipetted 2 micro l from samples 04, 05, 06, 07. Completed at 12:47.</p></li>
<li><p>FAB pipetted 2 micro l from samples 08, 09, 10, 11. Completed at 12:50</p></li>
<li><p>JRE vortexed each sample briefly. Completed at 12:52.</p>
<p>6. Incubate the prepared samples at room temperature for 5 minutes, protected from light.</p></li>
<li><p>Finished incubating at 12:58.</p>
<p>7. Select the dsDNA protocol on the <strong>Quantus Fluorometer</strong>.</p></li>
<li><p>Completed by ELP at .</p>
<p>8. Prior to running samples on the Quantus Fluorometer, you should blend them well by <strong>vortexing</strong> briefly and then spinning them down briefly using the <strong>microcentrifuge</strong>. You should also wipe down the sides of the tubes with <strong>kimwipes</strong> to make sure the sides are clear from smudges.&nbsp;</p></li>
<li><p>ELP vortexed and centrifuged each sample 2 at a time while JRE placed in Quantus.</p>
<p>9. If needed (you will need it!), calibrate the <strong>Quantus Fluorometer</strong> by reading the blank (Step 3; this should read as 0 ng/μl) and standard (Step 4; this should read as 100 ng/μl) samples in the Calibration screen, then select “Save”.&nbsp;</p>
<ul>
<li><strong>Note</strong>: a few ng/μl away from what it should be is ok… but large deviations need to be troubleshooted!</li>
</ul></li>
<li><p>Calibration complete at 1:00.</p>
<p>10. Enter the volume of the unknown sample (1–2 μl used in Step 5) and desired concentration units (in our case, ng/μl).</p></li>
<li><p>Already set prior to starting</p>
<p>11. Pipette to mix each sample at least 10 times before putting the desired amount in the working solution.&nbsp;</p></li>
<li><p>Done</p>
<p>12. One sample at a time, mix by vortexing, briefly spin down and wipe the outside of the tube with a kimwipe. Measure fluorescence of the unknown sample and record the final sample concentration results.</p>
<p>BL read as: 0.08 ng/uL</p>
<p>Standard read as: 100 ng/uL</p></li>
</ul></li>
</ul>
<ol type="1">
<li>Clean pedestal w kimwipes and nuclease free water
<ul>
<li>ELP completed at 1:06.</li>
</ul></li>
<li>Blank w 2 micro l of nuclease free water or storage medium (Buffer ATE)
<ul>
<li>ELP blanked with Buffer ATE. Completed at 1:07</li>
</ul></li>
<li>Between each sample, clean w kim wipe and nuclease free water</li>
<li>2 micro l of sample for each run</li>
</ol></li>
</ul>
<table class="table">
<colgroup>
<col style="width: 14%">
<col style="width: 13%">
<col style="width: 7%">
<col style="width: 17%">
<col style="width: 12%">
<col style="width: 32%">
</colgroup>
<thead>
<tr class="header">
<th>Sample ID</th>
<th>Taxon</th>
<th>Label</th>
<th>Final Concentration (ng/𝜇l)</th>
<th><strong>A260/A280 value</strong></th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Control (sterile water)</td>
<td>NA</td>
<td>AGI24-DBCL</td>
<td>0.04 -Q 0.8-N</td>
<td>0.76</td>
<td></td>
</tr>
<tr class="even">
<td>GOMEZ-18-190</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB01</td>
<td>8.6 - Q 4.3-N</td>
<td>1.42</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-029</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB02</td>
<td>1.42 - Q -4.3 -N</td>
<td>1.16</td>
<td></td>
</tr>
<tr class="even">
<td>VOYT-18-017</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB03</td>
<td>2.2 - Q 2.4 -N</td>
<td>1.35</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-036</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB04</td>
<td>2.62 - Q 3.9 -N</td>
<td>1.71</td>
<td>Maybe not the entire 200𝜇l</td>
</tr>
<tr class="even">
<td>GIUNCHI-18-001</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB05</td>
<td>8.4 - Q 6.2 -N</td>
<td>1.83</td>
<td></td>
</tr>
<tr class="odd">
<td>DIFIORE-18-025</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB06</td>
<td>1.17 - Q 1.0 -N</td>
<td>0.19</td>
<td></td>
</tr>
<tr class="even">
<td>DIFIORE-18-026</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB07</td>
<td>5.2 - Q 5.6 -N</td>
<td>1.85</td>
<td></td>
</tr>
<tr class="odd">
<td>DIFIORE-18-027</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB08</td>
<td>0.436 - Q 3.3 -N</td>
<td>1.67</td>
<td></td>
</tr>
<tr class="even">
<td>DIFIORE-18-031</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB09</td>
<td><p>1.33 -Q</p>
<p>2.2-N</p></td>
<td>1.79</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-006</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB10</td>
<td><p>7.2 -Q</p>
<p>9.0-N</p></td>
<td>1.78</td>
<td>Pipetted most of 200 then a tiny bit extra some chunks</td>
</tr>
<tr class="even">
<td>DIFIORE-18-032</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB11</td>
<td><p>1.77 -Q</p>
<p>2.7-N</p></td>
<td>1.40</td>
<td>Slightly less buffer was added</td>
</tr>
</tbody>
</table>
</section>
</section>
<section id="mar-25---mar-31" class="level1">
<h1>Mar 25 - Mar 31</h1>
<section id="march-28-2024-post-aaba-re-quantitating" class="level2">
<h2 class="anchored" data-anchor-id="march-28-2024-post-aaba-re-quantitating">March 28, 2024 (Post AABA re-quantitating)</h2>
<ul>
<li><p>1. If necessary, prepare <strong>1X TE Buffer</strong>: to do this, you may need to dilute <strong>20X TE Buffer</strong> 20-fold with <strong>nuclease-free water</strong>. We do also have 1X TE BUffer available in the cabinet above the Centrifuge Workstation; there may also be pre-prepared 1X TE Buffer in the Underbench Fridge.</p></li>
<li><p>JRE mixed 10% bleach at 4:30, ethanol water bleach wiped.</p></li>
<li><p>JRE and ELP made and aliquotted 1X TE Buffer at 4:40.</p>
<ul>
<li><p>2. Prepare Working Solution: Dilute the <strong>QuantiFluor® dsDNA Dye</strong> 1:400 in <strong>1X TE buffer</strong>, and mix. Working Solution may be available in the Underbench Fridge.</p>
<ul>
<li><p>ELP and JRE made 1.2 ml (3 parts dye to 1,197 TEx1 three times) Working solution following protocol at 12:23.</p></li>
<li><p>JRE vortexed working solution at 12:31.</p></li>
<li><p>JRE added 200 micro l of working solution into 14 autoclaved 0.5 ml tubes. Completed at 5:18.</p>
<p>3. Prepare Blank sample: Mix 2 μl of <strong>1X TE buffer</strong> with 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> Working Solution in an empty <strong>0.5 ml PCR tube</strong>. <strong>Vortex</strong> well and protect tube from light (we do this by covering the tube with aluminum foil).</p></li>
<li><p>ELP pipetted 2 micro l of 1X TE buffer into 0.5ml tube at 5:10</p>
<p>4. Prepare Standard: Add 2 μl of the provided <strong>DNA Standard</strong> (100 ng/μl) to 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> working solution in an empty <strong>0.5 ml PCR tube</strong>. <strong>Vortex</strong> well and protect tube from light.</p></li>
<li><p>ELP pipetted 2 microl l of DNA Standard into 0.5 tube at 5:10</p>
<p>5. Prepare Samples to be quantified: Add 1 or 2 μl of unknown samples to 200 μl of <strong>QuantiFluor® dsDNA Dye</strong> working solution in <strong>0.5 ml PCR tubes</strong>. <strong>Vortex</strong> well, and protect tube from light. You will want to pipet the sample up and down multiple times to make sure the sample is well-blended.</p></li>
<li><p>JRE pipetted 2 micro l from all samples. Completed at 5:10.</p>
<p>6. Incubate the prepared samples at room temperature for 5 minutes, protected from light.</p></li>
<li><p>Finished incubating at 5:24.</p></li>
<li><p>Repipetted into smaller tubes starting at 5:25, ended at 5:35.</p>
<p>7. Select the dsDNA protocol on the <strong>Quantus Fluorometer</strong>.</p></li>
<li><p>Completed by JRE and ELP at 5:35.</p>
<p>8. Prior to running samples on the Quantus Fluorometer, you should blend them well by <strong>vortexing</strong> briefly and then spinning them down briefly using the <strong>microcentrifuge</strong>. You should also wipe down the sides of the tubes with <strong>kimwipes</strong> to make sure the sides are clear from smudges.&nbsp;</p></li>
<li><p>ELP vortexed and centrifuged each sample 2 at a time while JRE placed in Quantus.</p>
<p>9. If needed (you will need it!), calibrate the <strong>Quantus Fluorometer</strong> by reading the blank (Step 3; this should read as 0 ng/μl) and standard (Step 4; this should read as 100 ng/μl) samples in the Calibration screen, then select “Save”.&nbsp;</p>
<ul>
<li><strong>Note</strong>: a few ng/μl away from what it should be is ok… but large deviations need to be troubleshooted!</li>
</ul></li>
<li><p>Calibration complete at 5:36.</p>
<p>10. Enter the volume of the unknown sample (1–2 μl used in Step 5) and desired concentration units (in our case, ng/μl).</p></li>
<li><p>Already set prior to starting</p>
<p>11. Pipette to mix each sample at least 10 times before putting the desired amount in the working solution.&nbsp;</p></li>
<li><p>Done</p>
<p>12. One sample at a time, mix by vortexing, briefly spin down and wipe the outside of the tube with a kimwipe. Measure fluorescence of the unknown sample and record the final sample concentration results.</p>
<p>BL read as: Lower than contol</p>
<p>Standard read as: 100 ng/uL</p></li>
</ul></li>
</ul>
<ol type="1">
<li>Clean pedestal w kimwipes and nuclease free water
<ul>
<li>ELP completed at 1:06.</li>
</ul></li>
<li>Blank w 2 micro l of nuclease free water or storage medium (Buffer ATE)
<ul>
<li>ELP blanked with Buffer ATE. Completed at 1:07</li>
</ul></li>
<li>Between each sample, clean w kim wipe and nuclease free water</li>
<li>2 micro l of sample for each run</li>
</ol></li>
</ul>
<p>REDOING original quantitation!</p>
<table class="table">
<colgroup>
<col style="width: 14%">
<col style="width: 13%">
<col style="width: 7%">
<col style="width: 17%">
<col style="width: 12%">
<col style="width: 32%">
</colgroup>
<thead>
<tr class="header">
<th>Sample ID</th>
<th>Taxon</th>
<th>Label</th>
<th>Final Concentration (ng/𝜇l)</th>
<th><strong>A260/A280 value</strong></th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Control (sterile water)</td>
<td>NA</td>
<td>AGI24-DBCL</td>
<td>0.686 -Q 0.8-N</td>
<td>0.76</td>
<td></td>
</tr>
<tr class="even">
<td>GOMEZ-18-190</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB01</td>
<td>0.0012 - Q 4.3-N</td>
<td>1.42</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-029</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB02</td>
<td>0.921 - Q -4.3 -N</td>
<td>1.16</td>
<td></td>
</tr>
<tr class="even">
<td>VOYT-18-017</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB03</td>
<td>0.354 - Q 2.4 -N</td>
<td>1.35</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-036</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB04</td>
<td>0.860 - Q 3.9 -N</td>
<td>1.71</td>
<td>Maybe not the entire 200𝜇l</td>
</tr>
<tr class="even">
<td>GIUNCHI-18-001</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB05</td>
<td>1.79 - Q 6.2 -N</td>
<td>1.83</td>
<td></td>
</tr>
<tr class="odd">
<td>DIFIORE-18-025</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB06</td>
<td>0.711 - Q 1.0 -N</td>
<td>0.19</td>
<td></td>
</tr>
<tr class="even">
<td>DIFIORE-18-026</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB07</td>
<td>3.19 - Q 5.6 -N</td>
<td>1.85</td>
<td></td>
</tr>
<tr class="odd">
<td>DIFIORE-18-027</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB08</td>
<td>0.354 - Q 3.3 -N</td>
<td>1.67</td>
<td></td>
</tr>
<tr class="even">
<td>DIFIORE-18-031</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB09</td>
<td><p>0.292 -Q</p>
<p>2.2-N</p></td>
<td>1.79</td>
<td></td>
</tr>
<tr class="odd">
<td>VOYT-18-006</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB10</td>
<td><p>5.2 -Q</p>
<p>9.0-N</p></td>
<td>1.78</td>
<td>Pipetted most of 200 then a tiny bit extra some chunks</td>
</tr>
<tr class="even">
<td>DIFIORE-18-032</td>
<td>Lagothrix lagotricha</td>
<td>AGI24-DB11</td>
<td><p>0.821 -Q</p>
<p>2.7-N</p></td>
<td>1.40</td>
<td>Slightly less buffer was added</td>
</tr>
</tbody>
</table>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>