<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>Metabarcoding_Weekly</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="Metabarcoding_Weekly_files/libs/clipboard/clipboard.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/quarto.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/popper.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="Metabarcoding_Weekly_files/libs/quarto-html/anchor.min.js"></script>
<link href="Metabarcoding_Weekly_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="Metabarcoding_Weekly_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="Metabarcoding_Weekly_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Metabarcoding_Weekly</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Evelyn Pain, Fatimah Bouderdaben, and Jimmy Erkens </p>
          </div>
  </div>
    
  
    
  </div>
  

</header>

<section id="brief-github-protocol" class="level1">
<h1>Brief Github protocol</h1>
<p>JRE</p>
<ul>
<li><p>Install Git</p></li>
<li><p>Import in project via version control</p></li>
<li><p>You will have a project and all the files on Github will be pulled locally</p></li>
<li><p>Generate a personal access token on Github</p></li>
<li><p>Make your changes</p></li>
<li><p>Commit your changes (and write what you did in the commit box)</p></li>
<li><p>Stage the files you want to push up (this is just checking a box)</p>
<ul>
<li>If you need to pull anything, pull it (you‚Äôll be prompted)</li>
</ul></li>
<li><p>If there are any hiccups, follow the instructions in the R terminal</p></li>
<li><p>Push!</p></li>
<li><p>Make sure to pull before you get started in the future!</p></li>
</ul>
</section>
<section id="background-and-goals" class="level1">
<h1>Background and goals</h1>
<p>ELP</p>
<p>Our internship team has 4 main goals:</p>
<ol type="1">
<li><p>To gain familiarity with genomics/genetics literature and theory through the assigned readings</p></li>
<li><p>To get practice with the lab skills necessary for extraction, PCR and sequencing (with the Oxford Nanopore MinION)</p></li>
<li><p>To develop a working protocol to DNA metabarcode woolly fecal samples to detect insects (and plants?) in their diet</p></li>
<li><p>To practice bioinformatics skills by identifying and comparing the species eaten by YTWMs and common woolly monkeys</p></li>
</ol>
<p>ELP</p>
</section>
<section id="protocol-dneasy-powerlyzer-powersoil-kit" class="level1">
<h1>Protocol (<strong>DNeasy PowerLyzer PowerSoil Kit)</strong></h1>
<p>This protocol is modified from the protocol developed by Mel Zarate in the SMAGL lab following a template from Dr.&nbsp;Liz Mallott (Washington University in St.&nbsp;Louis)</p>
<section id="general-notes-before-starting" class="level2">
<h2 class="anchored" data-anchor-id="general-notes-before-starting">General notes before starting</h2>
<ol type="1">
<li><p>Ensure that you are wearing suitable PPE: a lab coat, protective eye glasses and gloves (you can consult the appropriate safety data sheets from product suppliers if you have any doubts)</p></li>
<li><p>Turn on the biosafety cabinet (BSC) and let it run for a minimum of 15 minutes before starting the protocol</p></li>
<li><p>Turn on the water bath and set it to 65 ¬∞C</p></li>
<li><p>Make up a new batch of 10% bleach solution (1 part bleach to 9 parts distilled water) for the day and dispose of any from previous days</p></li>
<li><p>Clean all pipettes and work surfaces (including inside the biosafety cabinet) first with the bleach solution, then distilled water, and finally with 70% ethanol</p></li>
<li><p>Ensure that you have sufficient supplies (i.e., autoclaved, sterile tubes, pipette tips, etc.) before beginning</p></li>
<li><p>UV all tubes, pipettes and pipette tips in either the BSC or the PCR hood for 10 minutes before starting (be careful to limit your exposure to the UV light!)</p></li>
<li><p>Removed samples to be extracted from the freezer and let them thaw on ice</p></li>
</ol>
</section>
<section id="notes-on-the-protocol" class="level2">
<h2 class="anchored" data-anchor-id="notes-on-the-protocol">Notes on the protocol</h2>
<ul>
<li><p>Make sure to label all tubes on both the top and sides (it is easy for labels to get rubbed off during the protocol)</p></li>
<li><p>Always include a control sample of sterile/DNAse free water for every extraction so that you can detect potential contamination!</p></li>
<li><p>Perform all centrifugation steps at room temperature (15‚Äì25 ¬∞C).</p></li>
<li><p>If <strong>Solution C1</strong> has precipitated, heat at 60 ¬∞C until the precipitate dissolves.</p></li>
<li><p>Shake to mix <strong>Solution C4</strong> before use.</p></li>
</ul>
</section>
<section id="required-equipment-and-supplies" class="level2">
<h2 class="anchored" data-anchor-id="required-equipment-and-supplies">Required equipment and supplies</h2>
<ul>
<li>In this protocol, equipment/reagents available in the DNeasy PowerLyzer PowerSoil Kit will be marked in <span style="color:blue">blue</span>. Equipment and supplies available in the SMAGL will be in <span style="color:green">green</span>.</li>
<li>You will use the following pieces of equipment: a <span style="color:green">biosafety cabinet</span>, <span style="color:green">fridge</span> (located below the <strong>PCR workstation</strong>), <span style="color:green">pipettes</span> (the extraction set are located inside the <strong>BSC</strong> or at the <strong>extraction workstation</strong>), <span style="color:green">microcentrifuge</span> (located at <strong>centrifuge workstation</strong>) set to 10,000 rcf (not rpm), a <span style="color:green">heating block</span> or <span style="color:green">waterbath</span> set to 65 ¬∞C (located at the <strong>extraction workstation</strong>), a <span style="color:green">vortexer</span> (located at the <strong>extraction workstation</strong> or inside <strong>BSC</strong>) and a <span style="color:green">bead-beater</span> (located in the <strong>BSC</strong>)</li>
<li>You will also need <span style="color:green">pipette tips</span> (located on shelves above the <strong>PCR</strong> or <strong>quantification workstations</strong>). ONLY use filter tips!</li>
</ul>
</section>
<section id="protocol" class="level2">
<h2 class="anchored" data-anchor-id="protocol">Protocol</h2>
<ol type="1">
<li>Label the same number of <span style="color:blue">PowerBead tubes</span> as samples, plus one for a control</li>
<li>Vortex the fecal sample for a few seconds</li>
<li>Add 0.25 g of fecal matter to <span style="color:blue">PowerBead tubes</span> (you will need to cut the tips off each pipette tip to make the point wide enough to be able to suck up the sample)</li>
<li>Add 250 ùúál of <span style="color:green">nuclease free water</span> to the control <span style="color:blue">PowerBead tube</span></li>
<li>Invert the <span style="color:blue">PowerBead solution</span> a few times to mix</li>
<li>Add 750 ùúál of <span style="color:blue">PowerBead solution</span> to each <span style="color:blue">PowerBead tube</span></li>
<li>Add 60 ùúál of <span style="color:blue">Solution C1</span> to each tube (if the solution has precipitated, heat the solution to 60 ¬∞C until the precipitate has dissolved in either a <span style="color:green">heating block</span> or <span style="color:green">water bath</span>)</li>
<li>Vortex each tube briefly</li>
<li>Incubate the tubes at 65 ¬∞C for 30 minutes in the <span style="color:green">water bath</span> or <span style="color:green">heating block</span> (the <span style="color:green">heating block</span> can be cleaned and placed in the <span style="color:green">BSC</span>)</li>
<li>Secure tubes in the <span style="color:green">Bead Beater</span> (located in the <span style="color:green">BSC</span>) and run for 5 minutes, pause for 30 seconds, and run again for 5 minutes (the original protocol says to run at 2000 RPM but our <span style="color:green">Bead Beater</span> only has one speed, which is 3000 RPM)</li>
<li>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 10,000 g (rcf) for 30 seconds (make sure that the centrifuge is <strong>balanced</strong>, the lid has been screwed closed and that the tubes can rotate freely in the centrifuge without rubbing)</li>
<li>Check whether the sample in each tube has pelleted, if it has not you can centrifuge samples again for 3 minutes at the same speed</li>
<li>Label clean 2 ml <span style="color:blue">Collection tubes</span> with the same numbers as the <span style="color:blue">PowerBead tubes</span></li>
<li>Pipette off the supernatant (~ 400-500 ùúál) off into the clean <span style="color:blue">Collection tubes</span> (there may still be some bits of fecal material in the supernatant at this stage)</li>
<li>Add 250 ùúál of <span style="color:blue">Solution C2</span> to each tube (this solution has Inhibitor Removal Technology; it contains a reagent that can precipitate non-DNA organic and inorganic material, including humic substances, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications)</li>
<li>Vortex tubes for 5 seconds</li>
<li>Incubate tubes at 4 ¬∞C for 10 minutes (you can use the old <span style="color:green">fridge</span>)</li>
<li>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 10,000 g (rcf) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>, the lid has been screwed closed and that the tubes can rotate freely in the centrifuge without rubbing)</li>
<li>Label a new set of clean 2 ml <span style="color:blue">Collection tubes</span></li>
<li>Pipette off up to 600 ùúál of the supernatant into the new set of <span style="color:blue">Collection tubes</span> (make sure you only transfer the liquid and not any of the pellet. The pellet at this point contains non-DNA organic and inorganic material, including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet)</li>
<li>Add 200 ùúál of <span style="color:blue">Solution C3</span> to each tube (this solution has Inhibitor Removal Technology and is a second reagent to precipitate additional non-DNA organic and inorganic material including humic acid, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications)</li>
<li>Vortex each tube briefly</li>
<li>Incubate tubes for 4 ¬∞C for 10 minutes (you can use the old <span style="color:green">fridge</span>)</li>
<li>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 10,000 g (rcf) for 1 minute (make sure that the centrifuge is <strong>balanced</strong>, the lid has been screwed closed and that the tubes can rotate freely in the centrifuge without rubbing)</li>
<li>Label a new set of clean 2 ml <span style="color:blue">Collection tubes</span></li>
<li>Pipette off up to 750 ùúál of the supernatant into the new set of <span style="color:blue">Collection tubes</span> (make sure you only transfer the liquid and not any of the pellet. The pellet at this point contains non-DNA organic and inorganic material, including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet)</li>
<li>Shake <span style="color:blue">Solution C4</span> to mix (this solution containsguanidine salts,which are <strong>highly reactive. Do not mix or clean with bleach!!!</strong> This solution is a high-concentration salt solution. Since DNA binds tightly to silica at high salt concentrations, this will adjust the DNA solution salt concentrations to allow binding of DNA, but not non-DNA organic and inorganic material that may still be present at low levels, to the <span style="color:blue">Spin Filter</span>)</li>
<li>Add 1.2 ml of <span style="color:blue">Solution C4</span> to each tube (this will fill the tube to the brim so pipette cautiously)</li>
<li>Vortex each tube for 5 seconds</li>
<li>Label clean <span style="color:blue">Spin Filter tubes</span></li>
<li>Pipette 675 ùúál of the supernatant into the <span style="color:blue">Spin Filter tubes</span> making sure not to touch the filter with the pipette tip (DNA is selectively bound to the silica membrane in the MB Spin Column device in the high salt solution. Contaminants pass through the filter membrane, leaving only DNA bound to the membrane)</li>
<li>Centrifuge the <span style="color:blue">Spin Filter tubes</span> in the <span style="color:green">microcentrifuge</span> at 10,000 g (rcf) for 1 minute (make sure that the centrifuge is <strong>balanced</strong> and the lid has been screwed closed)</li>
<li>Discard the liquid flow through in the <span style="color:blue">Spin Filter tubes</span> (to do this pull the filter portion of the tube up and pour the liquid out of the lower portion of the tube. Put the tube back together)</li>
<li>Repeat steps 31 to 33 until all the supernatant has been passed through the <span style="color:blue">Spin Filter tubes</span> (this will take 3x through the column)</li>
<li>Add 500 ùúál of <span style="color:blue">Solution C5</span> to each <span style="color:blue">Spin Filter tube</span> making sure not to touch the filter with the pipette tip (this solution is an ethanol-based wash solution used to further clean the DNA that is bound to the silica filter membrane in the <span style="color:blue">Spin Filter</span>. This wash solution removes residual salt, humic acid, and other contaminants while allowing the DNA to stay bound to the silica membrane)</li>
<li>Centrifuge the <span style="color:blue">Spin Filter tubes</span> in the <span style="color:green">microcentrifuge</span> at 10,000 g (rcf) for 30 seconds (make sure that the centrifuge is <strong>balanced</strong> and the lid has been screwed closed)</li>
<li>Discard the liquid flow through in the <span style="color:blue">Spin Filter tubes</span> (to do this pull the filter portion of the tube up and pour the liquid out of the lower portion of the tube. Put the tube back together)</li>
<li>Centrifuge the <span style="color:blue">Spin Filter tubes</span> a second time in the <span style="color:green">microcentrifuge</span> at 10,000 g (rcf) for 1 minute (make sure that the centrifuge is <strong>balanced</strong> and the lid has been screwed closed. The second spin removes residual <span style="color:blue">Solution C5</span>. It is critical to remove all traces of wash solution because the ethanol in this solution can interfere with many downstream DNA applications, such as PCR and gel electrophoresis)</li>
<li>Label a new set of clean 2 ml <span style="color:blue">Collection tubes</span> on both the top and sides! (These are the tubes going into the freezer, so labeling is extra important!)</li>
<li>Place the <span style="color:blue">Spin Filters</span> into these new <span style="color:blue">Collection tubes</span> (to do this pull the filter portion of the tube up and place in the new tubes). Avoid splashing any <span style="color:blue">Solution C5</span> into the <span style="color:blue">Spin Filters</span>)</li>
<li>Incubate <span style="color:blue">Solution C6</span> at 65 ¬∞C for 5 minutes in the <span style="color:green">water bath</span> or <span style="color:green">heating block</span></li>
<li>Add 100 ùúál of <span style="color:blue">Solution C6</span> to the center of each <span style="color:blue">Spin Filter</span> making sure not to touch the filter with the pipette tip (pipetting the solution in the center of the small white membrane will make sure the entire membrane is wet. This will result in a more efficient and complete release of the DNA from the <span style="color:blue">Spin Filter</span> membrane. As the solution passes through the silica membrane, DNA that was bound in the presence of high salt is selectively released by the solution, which lacks salt)</li>
<li>Incubate the tubes at room temperature for 25 minutes</li>
<li>Centrifuge the tubes in the <span style="color:green">microcentrifuge</span> at 10,000 g (rcf) for 30 seconds (make sure that the centrifuge is <strong>balanced</strong> and the lid has been screwed closed. This time each tube will have 2 lids, only close the inner lid and then place the samples in the centrifuge in pairs with an empty space between each pair. Cross the outer (open) lids of each pair trying to avoid the lids touching each other)</li>
<li>Pull the filter portion of the tube up and cap the tube (you can either discard it or save them if you want to try and get more DNA out of them)</li>
<li>The liquid in the tube contains the DNA from the samples and should be stored in the <span style="color:green">-20 ¬∞C freezer</span> when not in use! (Be sure to record where in the freezer you‚Äôve placed them, and make sure that they are all together in a single, clearly labeled container/rack)</li>
<li>The next step is to quantify the DNA in the samples (this is done by measuring the dsDNA concentration using the <span style="color:green">Quantus</span> and the sample A260/A280 using the <span style="color:green">Nanodrop</span>, both are located at the <strong>Extraction Workstation</strong>). The results should be recorded in your lab notebook</li>
</ol>
<pre><code>- &lt;input type="checkbox" unchecked&gt; Venus&lt;/input&gt;
- &lt;input type="checkbox" checked&gt; Mars&lt;/input&gt;</code></pre>
</section>
</section>
<section id="feb-12---feb-18" class="level1">
<h1>Feb 12 - Feb 18</h1>
<p>JRE</p>
<p>This was the week we were supposed to start the extraction protocol, but got postponed.</p>
<p>Fati: Practice Pipetting</p>
</section>
<section id="feb-20---feb-25" class="level1">
<h1>Feb 20 - Feb 25</h1>
<p><input type="checkbox" class="checkbox"></p>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "Óßã";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>